<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Querying and Managing Large Biodiversity Occurrence Datasets â€¢ occCite</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="Querying and Managing Large Biodiversity Occurrence Datasets">
<meta property="og:description" content="Facilitates the gathering of biodiversity occurrence data from disparate sources. Metadata is managed throughout the process to facilitate reporting and enhanced ability to repeat analyses.">
<meta property="og:image" content="/reference/figures/occTag.png">
<meta property="og:image:alt" content="Gray specimen tag that says 'occCite: Tools for querying and citing large occurrence datasets'">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:creator" content="@hannahOish">
<meta name="twitter:site" content="@hannahOish">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">occCite</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.4.7</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="articles/occCite_Vignette.html">An overview of executing searches and generating citations using occCite</a>
    </li>
  </ul>
</li>
<li>
  <a href="news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/hannahlowens/occCite/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">


<div id="introduction" class="section level1">
<div class="page-header"><h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1></div>
<p>We have entered the age of data-intensive scientific discovery. As data sets increase in complexity and heterogeneity, we must preserve the cycle of data citation from primary data sources to aggregating databases to research products and back to primary data sources. The citation cycle keeps science transparent, but it is also key to supporting primary providers by documenting the use of their data. The Global Biodiversity Information Facility and other data aggregators have made great strides in harvesting citation data from research products and linking them back to primary data providers. However, this only works if those that publish research products cite primary data sources in the first place. We developed <code>occCite</code>, a set of <code>R</code>-based tools for downloading, managing, and citing biodiversity data, to advance toward the goal of closing the data provenance cycle. These tools preserve links between occurrence data and primary providers once researchers download aggregated data, and facilitate the citation of primary data providers in research papers.</p>
<p>The <code>occCite</code> work flow follows a three-step process. First, the user inputs one or more taxonomic names (or a phylogeny). <code>occCite</code> then rectifies those names by checking them against one or more taxonomic databases, which can be specified by the user (see the <a href="http://gni.globalnames.org/%7D">Global Names List</a>). The results of the taxonomic rectification are then kept in an <code>occCiteData</code> object in local memory. Next, <code>occCite</code> takes the <code>occCiteData</code> object and user-defined search parameters to query BIEN (through <code>rbien</code>) and/or GBIF(through <code>rGBIF</code>) for records. The results are appended to the <code>occCiteData</code> object, along with metadata on the search. Finally, the user can pass the <code>occCiteData</code> object to <code>occCitation</code>, which compiles citations for the primary providers, database aggregators, and <code>R</code> packages used to build the data set.</p>
<p>For an overview tutorial video of the package, see <a href="https://www.youtube.com/watch?v=7qSCULN_VjY&amp;t=17s">our YouTube video</a>.</p>
<p>Future iterations of <code>occCite</code> will track citation data through the data cleaning process and provide a series of visualizations on raw query results and final datasets. It will also provide data citations in a format congruent with best-practice recommendations for large biodiversity datasets. Based on these data citation tools, we will also propose a new set of standards for citing primary biodiversity data in published research articles that provides due credit to contributors and allows them to track the use of their work. Keep checking back!</p>
</div>
<div id="setup" class="section level1">
<h1 class="hasAnchor">
<a href="#setup" class="anchor"></a>Setup</h1>
<p>If you plan to query GBIF, you will need to provide them with your user login information. We have provided a dummy login below to show you the format. <em>You will need to provide actual account information.</em> This is because you will actually be downloading <em>all</em> of the records available for the species using <code>occ_download()</code>, instead of getting results from <code>occ_search()</code>, which has a hard limit of 200,000 occurrences.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/hannahlowens/occCite">occCite</a></span><span class="op">)</span>;
<span class="co">#Creating a GBIF login</span>
<span class="va">GBIFLogin</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/GBIFLoginManager.html">GBIFLoginManager</a></span><span class="op">(</span>user <span class="op">=</span> <span class="st">"occCiteTester"</span>,
                          email <span class="op">=</span> <span class="st">"****@yahoo.com"</span>,
                          pwd <span class="op">=</span> <span class="st">"12345"</span><span class="op">)</span></code></pre></div>
</div>
<div id="performing-a-simple-search" class="section level1">
<h1 class="hasAnchor">
<a href="#performing-a-simple-search" class="anchor"></a>Performing a simple search</h1>
<div id="the-basics" class="section level2">
<h2 class="hasAnchor">
<a href="#the-basics" class="anchor"></a>The basics</h2>
<p>At its simplest, <code>occCite</code> allows you to search for occurrences for a single species. The taxonomy of the user-specified species will be verified using EOL and NCBI taxonomies by default.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Simple search</span>
<span class="va">mySimpleOccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/occQuery.html">occQuery</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Protea cynaroides"</span>,
                            datasources <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"gbif"</span>, <span class="st">"bien"</span><span class="op">)</span>,
                            GBIFLogin <span class="op">=</span> <span class="va">GBIFLogin</span>, 
                            GBIFDownloadDirectory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/find.package.html">path.package</a></span><span class="op">(</span><span class="st">"occCite"</span><span class="op">)</span>, <span class="st">"/extdata/"</span><span class="op">)</span>,
                            checkPreviousGBIFDownload <span class="op">=</span> <span class="cn">T</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb3"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1"></a><span class="co">## Warning in taxonRectification(taxName = targets[[count]], datasources =</span></span>
<span id="cb3-2"><a href="#cb3-2"></a><span class="co">## datasources): Following sources not found in Global Names Index source list:</span></span>
<span id="cb3-3"><a href="#cb3-3"></a><span class="co">## NCBI</span></span>
<span id="cb3-4"><a href="#cb3-4"></a></span>
<span id="cb3-5"><a href="#cb3-5"></a><span class="co">## Warning in taxonRectification(taxName = targets[[count]], datasources =</span></span>
<span id="cb3-6"><a href="#cb3-6"></a><span class="co">## datasources): No valid taxonomic data sources supplied. Populating default list</span></span>
<span id="cb3-7"><a href="#cb3-7"></a><span class="co">## from all available sources.</span></span></code></pre></div>
<p>Here is what the GBIF results look like:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># GBIF search results</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">mySimpleOccCiteObject</span><span class="op">@</span><span class="va">occResults</span><span class="op">$</span><span class="va">`Protea cynaroides`</span><span class="op">$</span><span class="va">GBIF</span><span class="op">$</span><span class="va">OccurrenceTable</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1"></a><span class="co">##                name longitude  latitude day month year</span></span>
<span id="cb5-2"><a href="#cb5-2"></a><span class="co">## 1 Protea cynaroides  18.40540 -33.95891   4     1 2015</span></span>
<span id="cb5-3"><a href="#cb5-3"></a><span class="co">## 2 Protea cynaroides  18.42350 -33.96619  20     6 2019</span></span>
<span id="cb5-4"><a href="#cb5-4"></a><span class="co">## 3 Protea cynaroides  22.99340 -34.05478  16     6 2019</span></span>
<span id="cb5-5"><a href="#cb5-5"></a><span class="co">## 4 Protea cynaroides  18.40232 -34.08405   9     6 2019</span></span>
<span id="cb5-6"><a href="#cb5-6"></a><span class="co">## 5 Protea cynaroides  19.44807 -34.52123  13     6 2019</span></span>
<span id="cb5-7"><a href="#cb5-7"></a><span class="co">## 6 Protea cynaroides  18.39757 -34.07418   9     6 2019</span></span>
<span id="cb5-8"><a href="#cb5-8"></a><span class="co">##                                   Dataset                           DatasetKey</span></span>
<span id="cb5-9"><a href="#cb5-9"></a><span class="co">## 1 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb5-10"><a href="#cb5-10"></a><span class="co">## 2 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb5-11"><a href="#cb5-11"></a><span class="co">## 3 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb5-12"><a href="#cb5-12"></a><span class="co">## 4 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb5-13"><a href="#cb5-13"></a><span class="co">## 5 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb5-14"><a href="#cb5-14"></a><span class="co">## 6 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb5-15"><a href="#cb5-15"></a><span class="co">##   DataService</span></span>
<span id="cb5-16"><a href="#cb5-16"></a><span class="co">## 1        GBIF</span></span>
<span id="cb5-17"><a href="#cb5-17"></a><span class="co">## 2        GBIF</span></span>
<span id="cb5-18"><a href="#cb5-18"></a><span class="co">## 3        GBIF</span></span>
<span id="cb5-19"><a href="#cb5-19"></a><span class="co">## 4        GBIF</span></span>
<span id="cb5-20"><a href="#cb5-20"></a><span class="co">## 5        GBIF</span></span>
<span id="cb5-21"><a href="#cb5-21"></a><span class="co">## 6        GBIF</span></span></code></pre></div>
<p>And here are the BIEN results:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#BIEN search results</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">mySimpleOccCiteObject</span><span class="op">@</span><span class="va">occResults</span><span class="op">$</span><span class="va">`Protea cynaroides`</span><span class="op">$</span><span class="va">BIEN</span><span class="op">$</span><span class="va">OccurrenceTable</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb7"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1"></a><span class="co">##                name longitude latitude day month year Dataset DatasetKey</span></span>
<span id="cb7-2"><a href="#cb7-2"></a><span class="co">## 1 Protea cynaroides    22.875  -33.875  20     8 1973   SANBI       2249</span></span>
<span id="cb7-3"><a href="#cb7-3"></a><span class="co">## 2 Protea cynaroides    25.125  -33.875   3     7 1934   SANBI       2249</span></span>
<span id="cb7-4"><a href="#cb7-4"></a><span class="co">## 3 Protea cynaroides    20.375  -33.875  16     8 1952   SANBI       2249</span></span>
<span id="cb7-5"><a href="#cb7-5"></a><span class="co">## 4 Protea cynaroides    21.375  -33.375  20     3 1947   SANBI       2249</span></span>
<span id="cb7-6"><a href="#cb7-6"></a><span class="co">## 5 Protea cynaroides    20.875  -34.125  21     6 1987   SANBI       2249</span></span>
<span id="cb7-7"><a href="#cb7-7"></a><span class="co">## 6 Protea cynaroides    24.625  -33.625  12     9 1973   SANBI       2249</span></span>
<span id="cb7-8"><a href="#cb7-8"></a><span class="co">##   DataService</span></span>
<span id="cb7-9"><a href="#cb7-9"></a><span class="co">## 1        BIEN</span></span>
<span id="cb7-10"><a href="#cb7-10"></a><span class="co">## 2        BIEN</span></span>
<span id="cb7-11"><a href="#cb7-11"></a><span class="co">## 3        BIEN</span></span>
<span id="cb7-12"><a href="#cb7-12"></a><span class="co">## 4        BIEN</span></span>
<span id="cb7-13"><a href="#cb7-13"></a><span class="co">## 5        BIEN</span></span>
<span id="cb7-14"><a href="#cb7-14"></a><span class="co">## 6        BIEN</span></span></code></pre></div>
<p>There is also a summary method for <code>occCite</code> objects with some basic information about your search.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">mySimpleOccCiteObject</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb9"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1"></a><span class="co">##  </span></span>
<span id="cb9-2"><a href="#cb9-2"></a><span class="co">##  OccCite query occurred on: 05 March, 2021</span></span>
<span id="cb9-3"><a href="#cb9-3"></a><span class="co">##  </span></span>
<span id="cb9-4"><a href="#cb9-4"></a><span class="co">##  User query type: User-supplied list of taxa.</span></span>
<span id="cb9-5"><a href="#cb9-5"></a><span class="co">##  </span></span>
<span id="cb9-6"><a href="#cb9-6"></a><span class="co">##  Sources for taxonomic rectification: NCBI</span></span>
<span id="cb9-7"><a href="#cb9-7"></a><span class="co">##      </span></span>
<span id="cb9-8"><a href="#cb9-8"></a><span class="co">##  </span></span>
<span id="cb9-9"><a href="#cb9-9"></a><span class="co">##  Taxonomic cleaning results:     </span></span>
<span id="cb9-10"><a href="#cb9-10"></a><span class="co">## </span></span>
<span id="cb9-11"><a href="#cb9-11"></a><span class="co">##          Input Name        Best Match</span></span>
<span id="cb9-12"><a href="#cb9-12"></a><span class="co">## 1 Protea cynaroides Protea cynaroides</span></span>
<span id="cb9-13"><a href="#cb9-13"></a><span class="co">##                                                                                                                                     Taxonomic Databases w/ Matches</span></span>
<span id="cb9-14"><a href="#cb9-14"></a><span class="co">## 1 National Center for Biotechnology Information; Freebase; Encyclopedia of Life; CU*STAR; uBio NameBank; Arctos; Open Tree of Life Reference Taxonomy; iNaturalist</span></span>
<span id="cb9-15"><a href="#cb9-15"></a><span class="co">##  </span></span>
<span id="cb9-16"><a href="#cb9-16"></a><span class="co">##  Sources for occurrence data: gbif, bien</span></span>
<span id="cb9-17"><a href="#cb9-17"></a><span class="co">##      </span></span>
<span id="cb9-18"><a href="#cb9-18"></a><span class="co">##             Species Occurrences Sources</span></span>
<span id="cb9-19"><a href="#cb9-19"></a><span class="co">## 1 Protea cynaroides         828      15</span></span>
<span id="cb9-20"><a href="#cb9-20"></a><span class="co">##  </span></span>
<span id="cb9-21"><a href="#cb9-21"></a><span class="co">##  GBIF dataset DOIs:  </span></span>
<span id="cb9-22"><a href="#cb9-22"></a><span class="co">## </span></span>
<span id="cb9-23"><a href="#cb9-23"></a><span class="co">##             Species GBIF Access Date           GBIF DOI</span></span>
<span id="cb9-24"><a href="#cb9-24"></a><span class="co">## 1 Protea cynaroides       2019-07-15 10.15468/dl.iqnra2</span></span></code></pre></div>
</div>
<div id="simple-citations" class="section level2">
<h2 class="hasAnchor">
<a href="#simple-citations" class="anchor"></a>Simple citations</h2>
<p>After doing a search for occurrence points, you can use <code><a href="reference/occCitation.html">occCitation()</a></code> to generate citations for primary biodiversity databases, as well as database aggregators. <strong>Note:</strong> Currently, GBIF and BIEN are the only aggregators for which citations are supported.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Get citations</span>
<span class="va">mySimpleOccCitations</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/occCitation.html">occCitation</a></span><span class="op">(</span><span class="va">mySimpleOccCiteObject</span><span class="op">)</span></code></pre></div>
<p>Here is a simple way of generating a formatted citation document from the results of <code><a href="reference/occCitation.html">occCitation()</a></code>.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">mySimpleOccCitations</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb12"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb12-2"><a href="#cb12-2"></a><span class="co">## 'occCite'</span></span>
<span id="cb12-3"><a href="#cb12-3"></a></span>
<span id="cb12-4"><a href="#cb12-4"></a><span class="co">## Writing 5 Bibtex entries ... OK</span></span>
<span id="cb12-5"><a href="#cb12-5"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb12-6"><a href="#cb12-6"></a></span>
<span id="cb12-7"><a href="#cb12-7"></a><span class="co">## Warning: `as_data_frame()` was deprecated in tibble 2.0.0.</span></span>
<span id="cb12-8"><a href="#cb12-8"></a><span class="co">## Please use `as_tibble()` instead.</span></span>
<span id="cb12-9"><a href="#cb12-9"></a><span class="co">## The signature and semantics have changed, see `?as_tibble`.</span></span>
<span id="cb12-10"><a href="#cb12-10"></a></span>
<span id="cb12-11"><a href="#cb12-11"></a><span class="co">## Cameron E, Auckland Museum A M (2021). Auckland Museum Botany Collection. Version 1.57. Auckland War Memorial Museum. https://doi.org/10.15468/mnjkvv. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-12"><a href="#cb12-12"></a><span class="co">## Capers R (2014). CONN. University of Connecticut. https://doi.org/10.15468/w35jmd. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-13"><a href="#cb12-13"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb12-14"><a href="#cb12-14"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb12-15"><a href="#cb12-15"></a><span class="co">## Fatima Parker-Allie, Ranwashe F (2018). PRECIS. South African National Biodiversity Institute. https://doi.org/10.15468/rckmn2. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-16"><a href="#cb12-16"></a><span class="co">## Magill B, Solomon J, Stimmel H (2021). Tropicos Specimen Data. Missouri Botanical Garden. https://doi.org/10.15468/hja69f. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-17"><a href="#cb12-17"></a><span class="co">## Maitner, B. (2020). BIEN: Tools for Accessing the Botanical Information and Ecology. R package version 1.2.4. https://CRAN.R-project.org/package=BIEN.</span></span>
<span id="cb12-18"><a href="#cb12-18"></a><span class="co">## Missouri Botanical Garden,Herbarium. Accessed via BIEN on NA.</span></span>
<span id="cb12-19"><a href="#cb12-19"></a><span class="co">## MNHN, Chagnoux S (2021). The vascular plants collection (P) at the Herbarium of the MusÃ©um national d'Histoire Naturelle (MNHN - Paris). Version 69.199. MNHN - Museum national d'Histoire naturelle. https://doi.org/10.15468/nc6rxy. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-20"><a href="#cb12-20"></a><span class="co">## MNHN. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb12-21"><a href="#cb12-21"></a><span class="co">## naturgucker.de. naturgucker. https://doi.org/10.15468/uc1apo. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-22"><a href="#cb12-22"></a><span class="co">## NSW. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb12-23"><a href="#cb12-23"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb12-24"><a href="#cb12-24"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb12-25"><a href="#cb12-25"></a><span class="co">## Ranwashe F (2019). BODATSA: Botanical Collections. Version 1.4. South African National Biodiversity Institute. https://doi.org/10.15468/2aki0q. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-26"><a href="#cb12-26"></a><span class="co">## SANBI. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb12-27"><a href="#cb12-27"></a><span class="co">## Senckenberg (2020). African Plants - a photo guide. https://doi.org/10.15468/r9azth. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-28"><a href="#cb12-28"></a><span class="co">## Tela Botanica. Carnet en Ligne. https://doi.org/10.15468/rydcn2. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb12-29"><a href="#cb12-29"></a><span class="co">## UConn. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb12-30"><a href="#cb12-30"></a><span class="co">## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-15.</span></span></code></pre></div>
</div>
<div id="simple-taxonomic-rectification" class="section level2">
<h2 class="hasAnchor">
<a href="#simple-taxonomic-rectification" class="anchor"></a>Simple Taxonomic Rectification</h2>
<p>In the simplest of searches, such as the one above, the taxonomy of your input species name is automatically rectified through the <code>occCite</code> function <code><a href="reference/studyTaxonList.html">studyTaxonList()</a></code> using <code>gnr_resolve()</code> from the <code>taxize</code> <code>R</code> package. If you would like to change the source of the taxonomy being used to rectify your species names, you can specify as many taxonomic repositories as you like from the Global Names Index (GNI). The complete list of GNI repositories can be found <a href="http://gni.globalnames.org/data_sources">here</a>.</p>
<p><code><a href="reference/studyTaxonList.html">studyTaxonList()</a></code> chooses the taxonomic names closest to those being input and documents which taxonomic repositories agreed with those names. <code><a href="reference/studyTaxonList.html">studyTaxonList()</a></code> instantiates an <code>occCiteData</code> object the same way <code><a href="reference/occQuery.html">occQuery()</a></code> does. This object can be passed into <code><a href="reference/occQuery.html">occQuery()</a></code> to perform your occurrence data search.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Rectify taxonomy</span>
<span class="va">myTROccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/studyTaxonList.html">studyTaxonList</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Protea cynaroides"</span>, 
                                  datasources <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"National Center for Biotechnology Information"</span>, 
                                                  <span class="st">"Encyclopedia of Life"</span>, 
                                                  <span class="st">"Integrated Taxonomic Information SystemITIS"</span><span class="op">)</span><span class="op">)</span>
<span class="va">myTROccCiteObject</span><span class="op">@</span><span class="va">cleanedTaxonomy</span></code></pre></div>
<div class="sourceCode" id="cb14"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a><span class="co">##          Input Name        Best Match</span></span>
<span id="cb14-2"><a href="#cb14-2"></a><span class="co">## 1 Protea cynaroides Protea cynaroides</span></span>
<span id="cb14-3"><a href="#cb14-3"></a><span class="co">##                                        Taxonomic Databases w/ Matches</span></span>
<span id="cb14-4"><a href="#cb14-4"></a><span class="co">## 1 National Center for Biotechnology Information; Encyclopedia of Life</span></span></code></pre></div>
<hr>
</div>
</div>
<div id="advanced-features" class="section level1">
<h1 class="hasAnchor">
<a href="#advanced-features" class="anchor"></a>Advanced features</h1>
<div id="loading-data-from-previous-gbif-searches" class="section level2">
<h2 class="hasAnchor">
<a href="#loading-data-from-previous-gbif-searches" class="anchor"></a>Loading data from previous GBIF searches</h2>
<p>Querying GBIF can take quite a bit of time, especially for multiple species and/or well-known species. In this case, you may wish to access previously-downloaded data sets from your computer by specifying the general location of your downloaded <code>.zip</code> files. <code>occQuery</code> will crawl through your specified <code>GBIFDownloadDirectory</code> to collect all the <code>.zip</code> files contained in that folder and its subfolders. It will then import the most recent downloads that match your taxon list. These GBIF data will be appended to a BIEN search the same as if you do the simple real-time search (if you chose BIEN as well as GBIF), as was shown above. <code>checkPreviousGBIFDownload</code> is <code>TRUE</code> by default, but if <code>loadLocalGBIFDownload</code> is <code>TRUE</code>, <code>occQuery</code> will ignore <code>checkPreviousDownload</code>. It is also worth noting that <code>occCite</code> does not currently support mixed data download sources. That is, you cannot do GBIF queries for some taxa, download previously-prepared data sets for others, and load the rest from local data sets on your computer.</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Simple load</span>
<span class="va">myOldOccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/occQuery.html">occQuery</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Protea cynaroides"</span>, 
                                  datasources <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"gbif"</span>, <span class="st">"bien"</span><span class="op">)</span>, 
                                  GBIFLogin <span class="op">=</span> <span class="cn">NULL</span>, 
                                  GBIFDownloadDirectory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/find.package.html">path.package</a></span><span class="op">(</span><span class="st">"occCite"</span><span class="op">)</span>, <span class="st">"/extdata/"</span><span class="op">)</span>,
                                  loadLocalGBIFDownload <span class="op">=</span> <span class="cn">T</span>,
                                  checkPreviousGBIFDownload <span class="op">=</span> <span class="cn">F</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a><span class="co">## Warning in taxonRectification(taxName = targets[[count]], datasources =</span></span>
<span id="cb16-2"><a href="#cb16-2"></a><span class="co">## datasources): Following sources not found in Global Names Index source list:</span></span>
<span id="cb16-3"><a href="#cb16-3"></a><span class="co">## NCBI</span></span>
<span id="cb16-4"><a href="#cb16-4"></a></span>
<span id="cb16-5"><a href="#cb16-5"></a><span class="co">## Warning in taxonRectification(taxName = targets[[count]], datasources =</span></span>
<span id="cb16-6"><a href="#cb16-6"></a><span class="co">## datasources): No valid taxonomic data sources supplied. Populating default list</span></span>
<span id="cb16-7"><a href="#cb16-7"></a><span class="co">## from all available sources.</span></span></code></pre></div>
<p>Here is the result. Look familiar?</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#GBIF search results</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">myOldOccCiteObject</span><span class="op">@</span><span class="va">occResults</span><span class="op">$</span><span class="va">`Protea cynaroides`</span><span class="op">$</span><span class="va">GBIF</span><span class="op">$</span><span class="va">OccurrenceTable</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb18"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a><span class="co">##                name longitude  latitude day month year</span></span>
<span id="cb18-2"><a href="#cb18-2"></a><span class="co">## 1 Protea cynaroides  18.40540 -33.95891   4     1 2015</span></span>
<span id="cb18-3"><a href="#cb18-3"></a><span class="co">## 2 Protea cynaroides  18.42350 -33.96619  20     6 2019</span></span>
<span id="cb18-4"><a href="#cb18-4"></a><span class="co">## 3 Protea cynaroides  22.99340 -34.05478  16     6 2019</span></span>
<span id="cb18-5"><a href="#cb18-5"></a><span class="co">## 4 Protea cynaroides  18.40232 -34.08405   9     6 2019</span></span>
<span id="cb18-6"><a href="#cb18-6"></a><span class="co">## 5 Protea cynaroides  19.44807 -34.52123  13     6 2019</span></span>
<span id="cb18-7"><a href="#cb18-7"></a><span class="co">## 6 Protea cynaroides  18.39757 -34.07418   9     6 2019</span></span>
<span id="cb18-8"><a href="#cb18-8"></a><span class="co">##                                   Dataset                           DatasetKey</span></span>
<span id="cb18-9"><a href="#cb18-9"></a><span class="co">## 1 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb18-10"><a href="#cb18-10"></a><span class="co">## 2 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb18-11"><a href="#cb18-11"></a><span class="co">## 3 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb18-12"><a href="#cb18-12"></a><span class="co">## 4 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb18-13"><a href="#cb18-13"></a><span class="co">## 5 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb18-14"><a href="#cb18-14"></a><span class="co">## 6 iNaturalist research-grade observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7</span></span>
<span id="cb18-15"><a href="#cb18-15"></a><span class="co">##   DataService</span></span>
<span id="cb18-16"><a href="#cb18-16"></a><span class="co">## 1        GBIF</span></span>
<span id="cb18-17"><a href="#cb18-17"></a><span class="co">## 2        GBIF</span></span>
<span id="cb18-18"><a href="#cb18-18"></a><span class="co">## 3        GBIF</span></span>
<span id="cb18-19"><a href="#cb18-19"></a><span class="co">## 4        GBIF</span></span>
<span id="cb18-20"><a href="#cb18-20"></a><span class="co">## 5        GBIF</span></span>
<span id="cb18-21"><a href="#cb18-21"></a><span class="co">## 6        GBIF</span></span></code></pre></div>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#The full summary</span>
<span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">myOldOccCiteObject</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb20"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1"></a><span class="co">##  </span></span>
<span id="cb20-2"><a href="#cb20-2"></a><span class="co">##  OccCite query occurred on: 05 March, 2021</span></span>
<span id="cb20-3"><a href="#cb20-3"></a><span class="co">##  </span></span>
<span id="cb20-4"><a href="#cb20-4"></a><span class="co">##  User query type: User-supplied list of taxa.</span></span>
<span id="cb20-5"><a href="#cb20-5"></a><span class="co">##  </span></span>
<span id="cb20-6"><a href="#cb20-6"></a><span class="co">##  Sources for taxonomic rectification: NCBI</span></span>
<span id="cb20-7"><a href="#cb20-7"></a><span class="co">##      </span></span>
<span id="cb20-8"><a href="#cb20-8"></a><span class="co">##  </span></span>
<span id="cb20-9"><a href="#cb20-9"></a><span class="co">##  Taxonomic cleaning results:     </span></span>
<span id="cb20-10"><a href="#cb20-10"></a><span class="co">## </span></span>
<span id="cb20-11"><a href="#cb20-11"></a><span class="co">##          Input Name        Best Match</span></span>
<span id="cb20-12"><a href="#cb20-12"></a><span class="co">## 1 Protea cynaroides Protea cynaroides</span></span>
<span id="cb20-13"><a href="#cb20-13"></a><span class="co">##                                                                                                                                     Taxonomic Databases w/ Matches</span></span>
<span id="cb20-14"><a href="#cb20-14"></a><span class="co">## 1 National Center for Biotechnology Information; Freebase; Encyclopedia of Life; CU*STAR; uBio NameBank; Arctos; Open Tree of Life Reference Taxonomy; iNaturalist</span></span>
<span id="cb20-15"><a href="#cb20-15"></a><span class="co">##  </span></span>
<span id="cb20-16"><a href="#cb20-16"></a><span class="co">##  Sources for occurrence data: gbif, bien</span></span>
<span id="cb20-17"><a href="#cb20-17"></a><span class="co">##      </span></span>
<span id="cb20-18"><a href="#cb20-18"></a><span class="co">##             Species Occurrences Sources</span></span>
<span id="cb20-19"><a href="#cb20-19"></a><span class="co">## 1 Protea cynaroides         828      15</span></span>
<span id="cb20-20"><a href="#cb20-20"></a><span class="co">##  </span></span>
<span id="cb20-21"><a href="#cb20-21"></a><span class="co">##  GBIF dataset DOIs:  </span></span>
<span id="cb20-22"><a href="#cb20-22"></a><span class="co">## </span></span>
<span id="cb20-23"><a href="#cb20-23"></a><span class="co">##             Species GBIF Access Date           GBIF DOI</span></span>
<span id="cb20-24"><a href="#cb20-24"></a><span class="co">## 1 Protea cynaroides       2019-07-15 10.15468/dl.iqnra2</span></span></code></pre></div>
<p>Getting citation data works the exact same way with previously-downloaded data as it does from a fresh dataset.</p>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Get citations</span>
<span class="va">myOldOccCitations</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/occCitation.html">occCitation</a></span><span class="op">(</span><span class="va">myOldOccCiteObject</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">myOldOccCitations</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb22"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb22-2"><a href="#cb22-2"></a><span class="co">## 'occCite'</span></span>
<span id="cb22-3"><a href="#cb22-3"></a></span>
<span id="cb22-4"><a href="#cb22-4"></a><span class="co">## Writing 5 Bibtex entries ... OK</span></span>
<span id="cb22-5"><a href="#cb22-5"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb22-6"><a href="#cb22-6"></a></span>
<span id="cb22-7"><a href="#cb22-7"></a><span class="co">## Cameron E, Auckland Museum A M (2021). Auckland Museum Botany Collection. Version 1.57. Auckland War Memorial Museum. https://doi.org/10.15468/mnjkvv. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-8"><a href="#cb22-8"></a><span class="co">## Capers R (2014). CONN. University of Connecticut. https://doi.org/10.15468/w35jmd. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-9"><a href="#cb22-9"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb22-10"><a href="#cb22-10"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb22-11"><a href="#cb22-11"></a><span class="co">## Fatima Parker-Allie, Ranwashe F (2018). PRECIS. South African National Biodiversity Institute. https://doi.org/10.15468/rckmn2. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-12"><a href="#cb22-12"></a><span class="co">## Magill B, Solomon J, Stimmel H (2021). Tropicos Specimen Data. Missouri Botanical Garden. https://doi.org/10.15468/hja69f. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-13"><a href="#cb22-13"></a><span class="co">## Maitner, B. (2020). BIEN: Tools for Accessing the Botanical Information and Ecology. R package version 1.2.4. https://CRAN.R-project.org/package=BIEN.</span></span>
<span id="cb22-14"><a href="#cb22-14"></a><span class="co">## Missouri Botanical Garden,Herbarium. Accessed via BIEN on NA.</span></span>
<span id="cb22-15"><a href="#cb22-15"></a><span class="co">## MNHN, Chagnoux S (2021). The vascular plants collection (P) at the Herbarium of the MusÃ©um national d'Histoire Naturelle (MNHN - Paris). Version 69.199. MNHN - Museum national d'Histoire naturelle. https://doi.org/10.15468/nc6rxy. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-16"><a href="#cb22-16"></a><span class="co">## MNHN. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb22-17"><a href="#cb22-17"></a><span class="co">## naturgucker.de. naturgucker. https://doi.org/10.15468/uc1apo. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-18"><a href="#cb22-18"></a><span class="co">## NSW. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb22-19"><a href="#cb22-19"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb22-20"><a href="#cb22-20"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb22-21"><a href="#cb22-21"></a><span class="co">## Ranwashe F (2019). BODATSA: Botanical Collections. Version 1.4. South African National Biodiversity Institute. https://doi.org/10.15468/2aki0q. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-22"><a href="#cb22-22"></a><span class="co">## SANBI. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb22-23"><a href="#cb22-23"></a><span class="co">## Senckenberg (2020). African Plants - a photo guide. https://doi.org/10.15468/r9azth. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-24"><a href="#cb22-24"></a><span class="co">## Tela Botanica. Carnet en Ligne. https://doi.org/10.15468/rydcn2. Accessed via GBIF on 2019-07-15.</span></span>
<span id="cb22-25"><a href="#cb22-25"></a><span class="co">## UConn. https://doi.org/10.15468/dl.fpwlzt. Accessed via BIEN on 2018-08-14.</span></span>
<span id="cb22-26"><a href="#cb22-26"></a><span class="co">## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-15.</span></span></code></pre></div>
<p>Note that you can also load multiple species using either a vector of species names or a phylogeny (provided you have previously downloaded data for all of the species of interest), and you can load occurrences from non-GBIF data sources (e.g.Â BIEN) in the same query.</p>
<hr>
</div>
<div id="performing-a-multi-species-search" class="section level2">
<h2 class="hasAnchor">
<a href="#performing-a-multi-species-search" class="anchor"></a>Performing a Multi-Species Search</h2>
<p>In addition to doing a simple, single species search, you can also use <code>occCite</code> to search for and manage occurrence datasets for multiple species. You can either submit a vector of species names, or you can submit a <em>phylogeny</em>!</p>
</div>
<div id="occcite-with-a-phylogeny" class="section level2">
<h2 class="hasAnchor">
<a href="#occcite-with-a-phylogeny" class="anchor"></a>occCite with a Phylogeny</h2>
<p>Here is an example of how such a search is structured, using an unpublished phylogeny of billfishes.</p>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="http://ape-package.ird.fr/">ape</a></span><span class="op">)</span>
<span class="co">#Get tree</span>
<span class="va">treeFile</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span><span class="op">(</span><span class="st">"extdata/Fish_12Tax_time_calibrated.tre"</span>, package<span class="op">=</span><span class="st">'occCite'</span><span class="op">)</span>
<span class="va">phylogeny</span> <span class="op">&lt;-</span> <span class="fu">ape</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ape/man/read.nexus.html">read.nexus</a></span><span class="op">(</span><span class="va">treeFile</span><span class="op">)</span>
<span class="va">tree</span> <span class="op">&lt;-</span> <span class="fu">ape</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ape/man/drop.tip.html">extract.clade</a></span><span class="op">(</span><span class="va">phylogeny</span>, <span class="fl">18</span><span class="op">)</span>
<span class="co">#Query databases for names</span>
<span class="va">myPhyOccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/studyTaxonList.html">studyTaxonList</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">tree</span>, datasources <span class="op">=</span> <span class="st">"National Center for Biotechnology Information"</span><span class="op">)</span>
<span class="co">#Query GBIF for occurrence data</span>
<span class="va">myPhyOccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/occQuery.html">occQuery</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">myPhyOccCiteObject</span>, 
                            datasources <span class="op">=</span> <span class="st">"gbif"</span>,
                            GBIFDownloadDirectory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html">paste0</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/find.package.html">path.package</a></span><span class="op">(</span><span class="st">"occCite"</span><span class="op">)</span>, <span class="st">"/extdata/"</span><span class="op">)</span>, 
                            loadLocalGBIFDownload <span class="op">=</span> <span class="cn">T</span>,
                            checkPreviousGBIFDownload <span class="op">=</span> <span class="cn">F</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># What does a multispecies query look like?</span>
<span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb25"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1"></a><span class="co">##  </span></span>
<span id="cb25-2"><a href="#cb25-2"></a><span class="co">##  OccCite query occurred on: 05 March, 2021</span></span>
<span id="cb25-3"><a href="#cb25-3"></a><span class="co">##  </span></span>
<span id="cb25-4"><a href="#cb25-4"></a><span class="co">##  User query type: User-supplied phylogeny.</span></span>
<span id="cb25-5"><a href="#cb25-5"></a><span class="co">##  </span></span>
<span id="cb25-6"><a href="#cb25-6"></a><span class="co">##  Sources for taxonomic rectification: National Center for Biotechnology Information</span></span>
<span id="cb25-7"><a href="#cb25-7"></a><span class="co">##      </span></span>
<span id="cb25-8"><a href="#cb25-8"></a><span class="co">##  </span></span>
<span id="cb25-9"><a href="#cb25-9"></a><span class="co">##  Taxonomic cleaning results:     </span></span>
<span id="cb25-10"><a href="#cb25-10"></a><span class="co">## </span></span>
<span id="cb25-11"><a href="#cb25-11"></a><span class="co">##                   Input Name                 Best Match</span></span>
<span id="cb25-12"><a href="#cb25-12"></a><span class="co">## 1           Istiompax_indica           Istiompax indica</span></span>
<span id="cb25-13"><a href="#cb25-13"></a><span class="co">## 2             Kajikia_albida             Kajikia albida</span></span>
<span id="cb25-14"><a href="#cb25-14"></a><span class="co">## 3              Kajikia_audax              Kajikia audax</span></span>
<span id="cb25-15"><a href="#cb25-15"></a><span class="co">## 4 Tetrapturus_angustirostris Tetrapturus angustirostris</span></span>
<span id="cb25-16"><a href="#cb25-16"></a><span class="co">## 5         Tetrapturus_belone         Tetrapturus belone</span></span>
<span id="cb25-17"><a href="#cb25-17"></a><span class="co">## 6        Tetrapturus_georgii        Tetrapturus georgii</span></span>
<span id="cb25-18"><a href="#cb25-18"></a><span class="co">## 7      Tetrapturus_pfluegeri      Tetrapturus pfluegeri</span></span>
<span id="cb25-19"><a href="#cb25-19"></a><span class="co">##                  Taxonomic Databases w/ Matches</span></span>
<span id="cb25-20"><a href="#cb25-20"></a><span class="co">## 1 National Center for Biotechnology Information</span></span>
<span id="cb25-21"><a href="#cb25-21"></a><span class="co">## 2 National Center for Biotechnology Information</span></span>
<span id="cb25-22"><a href="#cb25-22"></a><span class="co">## 3 National Center for Biotechnology Information</span></span>
<span id="cb25-23"><a href="#cb25-23"></a><span class="co">## 4 National Center for Biotechnology Information</span></span>
<span id="cb25-24"><a href="#cb25-24"></a><span class="co">## 5 National Center for Biotechnology Information</span></span>
<span id="cb25-25"><a href="#cb25-25"></a><span class="co">## 6 National Center for Biotechnology Information</span></span>
<span id="cb25-26"><a href="#cb25-26"></a><span class="co">## 7 National Center for Biotechnology Information</span></span>
<span id="cb25-27"><a href="#cb25-27"></a><span class="co">##  </span></span>
<span id="cb25-28"><a href="#cb25-28"></a><span class="co">##  Sources for occurrence data: gbif</span></span>
<span id="cb25-29"><a href="#cb25-29"></a><span class="co">##      </span></span>
<span id="cb25-30"><a href="#cb25-30"></a><span class="co">##                      Species Occurrences Sources</span></span>
<span id="cb25-31"><a href="#cb25-31"></a><span class="co">## 1           Istiompax indica         468      23</span></span>
<span id="cb25-32"><a href="#cb25-32"></a><span class="co">## 2             Kajikia albida         167      16</span></span>
<span id="cb25-33"><a href="#cb25-33"></a><span class="co">## 3              Kajikia audax        6721      22</span></span>
<span id="cb25-34"><a href="#cb25-34"></a><span class="co">## 4 Tetrapturus angustirostris         174      22</span></span>
<span id="cb25-35"><a href="#cb25-35"></a><span class="co">## 5         Tetrapturus belone           9       6</span></span>
<span id="cb25-36"><a href="#cb25-36"></a><span class="co">## 6        Tetrapturus georgii          62       4</span></span>
<span id="cb25-37"><a href="#cb25-37"></a><span class="co">## 7      Tetrapturus pfluegeri         409       7</span></span>
<span id="cb25-38"><a href="#cb25-38"></a><span class="co">##  </span></span>
<span id="cb25-39"><a href="#cb25-39"></a><span class="co">##  GBIF dataset DOIs:  </span></span>
<span id="cb25-40"><a href="#cb25-40"></a><span class="co">## </span></span>
<span id="cb25-41"><a href="#cb25-41"></a><span class="co">##                      Species GBIF Access Date           GBIF DOI</span></span>
<span id="cb25-42"><a href="#cb25-42"></a><span class="co">## 1           Istiompax indica       2019-07-04 10.15468/dl.crapuf</span></span>
<span id="cb25-43"><a href="#cb25-43"></a><span class="co">## 2             Kajikia albida       2019-07-04 10.15468/dl.lnwf6a</span></span>
<span id="cb25-44"><a href="#cb25-44"></a><span class="co">## 3              Kajikia audax       2019-07-04 10.15468/dl.txromp</span></span>
<span id="cb25-45"><a href="#cb25-45"></a><span class="co">## 4 Tetrapturus angustirostris       2019-07-04 10.15468/dl.mumi5e</span></span>
<span id="cb25-46"><a href="#cb25-46"></a><span class="co">## 5         Tetrapturus belone       2019-07-04 10.15468/dl.q2nxb1</span></span>
<span id="cb25-47"><a href="#cb25-47"></a><span class="co">## 6        Tetrapturus georgii       2019-07-04 10.15468/dl.h860up</span></span>
<span id="cb25-48"><a href="#cb25-48"></a><span class="co">## 7      Tetrapturus pfluegeri       2019-07-04 10.15468/dl.qjidbs</span></span></code></pre></div>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Get citations</span>
<span class="va">myPhyOccCitations</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/occCitation.html">occCitation</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span><span class="op">)</span>

<span class="co">#Print citations as text with accession dates.</span>
<span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">myPhyOccCitations</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb27"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb27-2"><a href="#cb27-2"></a><span class="co">## 'occCite'</span></span>
<span id="cb27-3"><a href="#cb27-3"></a></span>
<span id="cb27-4"><a href="#cb27-4"></a><span class="co">## Writing 4 Bibtex entries ... OK</span></span>
<span id="cb27-5"><a href="#cb27-5"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb27-6"><a href="#cb27-6"></a></span>
<span id="cb27-7"><a href="#cb27-7"></a><span class="co">## Australian Museum (2021). Australian Museum provider for OZCAM. https://doi.org/10.15468/e7susi. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-8"><a href="#cb27-8"></a><span class="co">## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-9"><a href="#cb27-9"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s (RÃ©union). Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/elttrd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-10"><a href="#cb27-10"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s. Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/gdrknh. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-11"><a href="#cb27-11"></a><span class="co">## Bentley A (2021). KUBI Ichthyology Collection. Version 17.70. University of Kansas Biodiversity Institute. https://doi.org/10.15468/mgjasg. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-12"><a href="#cb27-12"></a><span class="co">## Bentley A (2021). KUBI Ichthyology Tissue Collection. Version 18.58. University of Kansas Biodiversity Institute. https://doi.org/10.15468/jmsnwg. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-13"><a href="#cb27-13"></a><span class="co">## Casassovici A, Brosens D (2021). Diveboard - Scuba diving citizen science observations. Version 54.34. Diveboard. https://doi.org/10.15468/tnjrgy. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-14"><a href="#cb27-14"></a><span class="co">## Catania D, Fong J (2021). CAS Ichthyology (ICH). Version 150.260. California Academy of Sciences. https://doi.org/10.15468/efh2ib. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-15"><a href="#cb27-15"></a><span class="co">## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-16"><a href="#cb27-16"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb27-17"><a href="#cb27-17"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb27-18"><a href="#cb27-18"></a><span class="co">## Chiang W (2014). Taiwan Fisheries Research Institute â€“ Digital archives of coastal and offshore specimens. TELDAP. https://doi.org/10.15468/xvxngy. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-19"><a href="#cb27-19"></a><span class="co">## Commonwealth Scientific and Industrial Research Organisation (2020). CSIRO Ichthyology provider for OZCAM. https://doi.org/10.15468/azp1pf. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-20"><a href="#cb27-20"></a><span class="co">## CSIRO - Arafura Sea shark surveys (Rachel cruises 1984-1985) https://doi.org/10.15468/yickr6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-21"><a href="#cb27-21"></a><span class="co">## CSIRO National Collections and Marine Infrastructure (NCMI) Information and Data Centre (IDC): Pelagic fish food web linkages, Queensland, Australia (2003-2005) https://doi.org/10.15468/yy5wdp. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-22"><a href="#cb27-22"></a><span class="co">## ElÃ­as GutiÃ©rrez M, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). CÃ³digos de barras de la vida en peces y zooplancton de MÃ©xico. Version 1.7. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/xmbkgo. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-23"><a href="#cb27-23"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-24"><a href="#cb27-24"></a><span class="co">## Feeney R (2019). LACM Vertebrate Collection. Version 18.7. Natural History Museum of Los Angeles County. https://doi.org/10.15468/77rmwd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-25"><a href="#cb27-25"></a><span class="co">## Frable B (2019). SIO Marine Vertebrate Collection. Version 1.7. Scripps Institution of Oceanography. https://doi.org/10.15468/ad1ovc. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-26"><a href="#cb27-26"></a><span class="co">## GonzÃ¡lez Acosta A F, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). AmpliaciÃ³n de la base de datos de la ictiofauna insular del Golfo de California. Version 1.7. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/p5ovq7. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-27"><a href="#cb27-27"></a><span class="co">## Grant S, McMahan C (2020). Field Museum of Natural History (Zoology) Fish Collection. Version 13.12. Field Museum. https://doi.org/10.15468/alz7wu. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-28"><a href="#cb27-28"></a><span class="co">## Harvard University M, Morris P J (2021). Museum of Comparative Zoology, Harvard University. Version 162.252. Museum of Comparative Zoology, Harvard University. https://doi.org/10.15468/p5rupv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-29"><a href="#cb27-29"></a><span class="co">## Millen B (2019). Ichthyology Collection - Royal Ontario Museum. Version 18.7. Royal Ontario Museum. https://doi.org/10.15468/syisbx. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-30"><a href="#cb27-30"></a><span class="co">## Ministry for Primary Industries (2014). New Zealand research tagging database. Southwestern Pacific OBIS, National Institute of Water and Atmospheric Research (NIWA), Wellington, New Zealand, 411926 records, Online http://nzobisipt.niwa.co.nz/resource.do?r=mpi_tag released on November 5, 2014. https://doi.org/10.15468/i66xdm. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-31"><a href="#cb27-31"></a><span class="co">## Museum and Art Gallery of the Northern Territory (2019). Northern Territory Museum and Art Gallery provider for OZCAM. https://doi.org/10.15468/giro3a. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-32"><a href="#cb27-32"></a><span class="co">## Museums Victoria (2021). Museums Victoria provider for OZCAM. https://doi.org/10.15468/lp1ctu. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-33"><a href="#cb27-33"></a><span class="co">## National Museum of Nature and Science, Japan (2020). Fish specimens of Kagoshima University Museum. https://doi.org/10.15468/vcj3j8. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-34"><a href="#cb27-34"></a><span class="co">## Orrell T, Informatics Office (2021). NMNH Extant Specimen Records. Version 1.41. National Museum of Natural History, Smithsonian Institution. https://doi.org/10.15468/hnhrg3. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-35"><a href="#cb27-35"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb27-36"><a href="#cb27-36"></a><span class="co">## Pozo de la Tijera M D C, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). Fortalecimiento de las colecciones de ECOSUR. Primera fase (Ictioplancton Chetumal). Version 1.3. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/orx3mk. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-37"><a href="#cb27-37"></a><span class="co">## Queensland Museum (2021). Queensland Museum provider for OZCAM. https://doi.org/10.15468/lotsye. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-38"><a href="#cb27-38"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb27-39"><a href="#cb27-39"></a><span class="co">## Raiva R, Santana P (2019). Diversidade e ocorrÃªncia de peixes em Inhambane (2009-2017). Version 1.4. National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/4fj2tq. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-40"><a href="#cb27-40"></a><span class="co">## Raiva R, Viador R, Santana P (2019). Diversidade e ocorrÃªncia de peixes na ZambÃ©zia (2003-2016). National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/mrz36h. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-41"><a href="#cb27-41"></a><span class="co">## Ranz J (2017). Banco de Datos de la Biodiversidad de la Comunitat Valenciana. Biodiversity data bank of Generalitat Valenciana. https://doi.org/10.15468/b4yqdy. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-42"><a href="#cb27-42"></a><span class="co">## Robins R (2021). UF FLMNH Ichthyology. Version 117.301. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-43"><a href="#cb27-43"></a><span class="co">## SÃ¡nchez GonzÃ¡lez S, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). TaxonomÃ­a y sistemÃ¡tica de la Ictiofauna de la BahÃ­a de Banderas del Estado de Nayarit, MÃ©xico. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/uhrwsl. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-44"><a href="#cb27-44"></a><span class="co">## South Australian Museum (2021). South Australian Museum Australia provider for OZCAM. https://doi.org/10.15468/wz4rrh. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-45"><a href="#cb27-45"></a><span class="co">## SWPRON (2014). Marine biological observation data from coastal and offshore surveys around New Zealand. Southwestern Pacific OBIS, National Institute of Water and Atmospheric Research (NIWA), Wellington, New Zealand, 9092 records, Online http://nzobisipt.niwa.co.nz/resource.do?r=mbis_nz released on January 16, 2018. https://doi.org/10.15468/pzpgop. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-46"><a href="#cb27-46"></a><span class="co">## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-47"><a href="#cb27-47"></a><span class="co">## Uchifune Y, Yamamoto H (2020). Asia-Pacific Dataset. Version 1.33. National Museum of Nature and Science, Japan. https://doi.org/10.15468/vjeh1p. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-48"><a href="#cb27-48"></a><span class="co">## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-49"><a href="#cb27-49"></a><span class="co">## UMS PatriNat (OFB-CNRS-MNHN), Paris (2020). DonnÃ©es d'occurrences EspÃ¨ces issues de l'inventaire des ZNIEFF. Version 1.1. https://doi.org/10.15468/ikshke. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-50"><a href="#cb27-50"></a><span class="co">## University of Washington Ichthyology Collection (UWFC) https://doi.org/10.15468/vvp7gr. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb27-51"><a href="#cb27-51"></a><span class="co">## Western Australian Museum (2019). Western Australian Museum provider for OZCAM. https://doi.org/10.15468/5qt0dm. Accessed via GBIF on 2019-07-04.</span></span></code></pre></div>
<p>It is also possible to print citations separated by species.</p>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">myPhyOccCitations</span>, bySpecies <span class="op">=</span> <span class="cn">T</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb29"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb29-2"><a href="#cb29-2"></a><span class="co">## 'occCite'</span></span>
<span id="cb29-3"><a href="#cb29-3"></a></span>
<span id="cb29-4"><a href="#cb29-4"></a><span class="co">## Writing 6 Bibtex entries ... OK</span></span>
<span id="cb29-5"><a href="#cb29-5"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb29-6"><a href="#cb29-6"></a></span>
<span id="cb29-7"><a href="#cb29-7"></a><span class="co">## Species: Istiompax indica </span></span>
<span id="cb29-8"><a href="#cb29-8"></a><span class="co">## </span></span>
<span id="cb29-9"><a href="#cb29-9"></a><span class="co">## Australian Museum (2021). Australian Museum provider for OZCAM. https://doi.org/10.15468/e7susi. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-10"><a href="#cb29-10"></a><span class="co">## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-11"><a href="#cb29-11"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s (RÃ©union). Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/elttrd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-12"><a href="#cb29-12"></a><span class="co">## Catania D, Fong J (2021). CAS Ichthyology (ICH). Version 150.260. California Academy of Sciences. https://doi.org/10.15468/efh2ib. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-13"><a href="#cb29-13"></a><span class="co">## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-14"><a href="#cb29-14"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb29-15"><a href="#cb29-15"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb29-16"><a href="#cb29-16"></a><span class="co">## CSIRO - Arafura Sea shark surveys (Rachel cruises 1984-1985) https://doi.org/10.15468/yickr6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-17"><a href="#cb29-17"></a><span class="co">## CSIRO National Collections and Marine Infrastructure (NCMI) Information and Data Centre (IDC): Pelagic fish food web linkages, Queensland, Australia (2003-2005) https://doi.org/10.15468/yy5wdp. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-18"><a href="#cb29-18"></a><span class="co">## ElÃ­as GutiÃ©rrez M, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). CÃ³digos de barras de la vida en peces y zooplancton de MÃ©xico. Version 1.7. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/xmbkgo. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-19"><a href="#cb29-19"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-20"><a href="#cb29-20"></a><span class="co">## Feeney R (2019). LACM Vertebrate Collection. Version 18.7. Natural History Museum of Los Angeles County. https://doi.org/10.15468/77rmwd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-21"><a href="#cb29-21"></a><span class="co">## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.</span></span>
<span id="cb29-22"><a href="#cb29-22"></a><span class="co">## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.</span></span>
<span id="cb29-23"><a href="#cb29-23"></a><span class="co">## Ministry for Primary Industries (2014). New Zealand research tagging database. Southwestern Pacific OBIS, National Institute of Water and Atmospheric Research (NIWA), Wellington, New Zealand, 411926 records, Online http://nzobisipt.niwa.co.nz/resource.do?r=mpi_tag released on November 5, 2014. https://doi.org/10.15468/i66xdm. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-24"><a href="#cb29-24"></a><span class="co">## Museum and Art Gallery of the Northern Territory (2019). Northern Territory Museum and Art Gallery provider for OZCAM. https://doi.org/10.15468/giro3a. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-25"><a href="#cb29-25"></a><span class="co">## Museums Victoria (2021). Museums Victoria provider for OZCAM. https://doi.org/10.15468/lp1ctu. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-26"><a href="#cb29-26"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb29-27"><a href="#cb29-27"></a><span class="co">## Pozo de la Tijera M D C, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). Fortalecimiento de las colecciones de ECOSUR. Primera fase (Ictioplancton Chetumal). Version 1.3. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/orx3mk. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-28"><a href="#cb29-28"></a><span class="co">## Queensland Museum (2021). Queensland Museum provider for OZCAM. https://doi.org/10.15468/lotsye. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-29"><a href="#cb29-29"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb29-30"><a href="#cb29-30"></a><span class="co">## Raiva R, Santana P (2019). Diversidade e ocorrÃªncia de peixes em Inhambane (2009-2017). Version 1.4. National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/4fj2tq. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-31"><a href="#cb29-31"></a><span class="co">## Raiva R, Viador R, Santana P (2019). Diversidade e ocorrÃªncia de peixes na ZambÃ©zia (2003-2016). National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/mrz36h. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-32"><a href="#cb29-32"></a><span class="co">## Robins R (2021). UF FLMNH Ichthyology. Version 117.301. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-33"><a href="#cb29-33"></a><span class="co">## SÃ¡nchez GonzÃ¡lez S, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). TaxonomÃ­a y sistemÃ¡tica de la Ictiofauna de la BahÃ­a de Banderas del Estado de Nayarit, MÃ©xico. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/uhrwsl. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-34"><a href="#cb29-34"></a><span class="co">## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-35"><a href="#cb29-35"></a><span class="co">## Uchifune Y, Yamamoto H (2020). Asia-Pacific Dataset. Version 1.33. National Museum of Nature and Science, Japan. https://doi.org/10.15468/vjeh1p. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-36"><a href="#cb29-36"></a><span class="co">## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-37"><a href="#cb29-37"></a><span class="co">## Western Australian Museum (2019). Western Australian Museum provider for OZCAM. https://doi.org/10.15468/5qt0dm. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-38"><a href="#cb29-38"></a></span>
<span id="cb29-39"><a href="#cb29-39"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb29-40"><a href="#cb29-40"></a><span class="co">## 'occCite'</span></span>
<span id="cb29-41"><a href="#cb29-41"></a></span>
<span id="cb29-42"><a href="#cb29-42"></a><span class="co">## Writing 6 Bibtex entries ... OK</span></span>
<span id="cb29-43"><a href="#cb29-43"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb29-44"><a href="#cb29-44"></a></span>
<span id="cb29-45"><a href="#cb29-45"></a><span class="co">## Species: Kajikia albida </span></span>
<span id="cb29-46"><a href="#cb29-46"></a><span class="co">## </span></span>
<span id="cb29-47"><a href="#cb29-47"></a><span class="co">## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-48"><a href="#cb29-48"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s. Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/gdrknh. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-49"><a href="#cb29-49"></a><span class="co">## Bentley A (2021). KUBI Ichthyology Collection. Version 17.70. University of Kansas Biodiversity Institute. https://doi.org/10.15468/mgjasg. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-50"><a href="#cb29-50"></a><span class="co">## Bentley A (2021). KUBI Ichthyology Tissue Collection. Version 18.58. University of Kansas Biodiversity Institute. https://doi.org/10.15468/jmsnwg. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-51"><a href="#cb29-51"></a><span class="co">## Casassovici A, Brosens D (2021). Diveboard - Scuba diving citizen science observations. Version 54.34. Diveboard. https://doi.org/10.15468/tnjrgy. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-52"><a href="#cb29-52"></a><span class="co">## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-53"><a href="#cb29-53"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb29-54"><a href="#cb29-54"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb29-55"><a href="#cb29-55"></a><span class="co">## ElÃ­as GutiÃ©rrez M, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). CÃ³digos de barras de la vida en peces y zooplancton de MÃ©xico. Version 1.7. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/xmbkgo. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-56"><a href="#cb29-56"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-57"><a href="#cb29-57"></a><span class="co">## Feeney R (2019). LACM Vertebrate Collection. Version 18.7. Natural History Museum of Los Angeles County. https://doi.org/10.15468/77rmwd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-58"><a href="#cb29-58"></a><span class="co">## Frable B (2019). SIO Marine Vertebrate Collection. Version 1.7. Scripps Institution of Oceanography. https://doi.org/10.15468/ad1ovc. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-59"><a href="#cb29-59"></a><span class="co">## Harvard University M, Morris P J (2021). Museum of Comparative Zoology, Harvard University. Version 162.252. Museum of Comparative Zoology, Harvard University. https://doi.org/10.15468/p5rupv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-60"><a href="#cb29-60"></a><span class="co">## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.</span></span>
<span id="cb29-61"><a href="#cb29-61"></a><span class="co">## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.</span></span>
<span id="cb29-62"><a href="#cb29-62"></a><span class="co">## Millen B (2019). Ichthyology Collection - Royal Ontario Museum. Version 18.7. Royal Ontario Museum. https://doi.org/10.15468/syisbx. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-63"><a href="#cb29-63"></a><span class="co">## Orrell T, Informatics Office (2021). NMNH Extant Specimen Records. Version 1.41. National Museum of Natural History, Smithsonian Institution. https://doi.org/10.15468/hnhrg3. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-64"><a href="#cb29-64"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb29-65"><a href="#cb29-65"></a><span class="co">## Pozo de la Tijera M D C, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). Fortalecimiento de las colecciones de ECOSUR. Primera fase (Ictioplancton Chetumal). Version 1.3. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/orx3mk. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-66"><a href="#cb29-66"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb29-67"><a href="#cb29-67"></a><span class="co">## Robins R (2021). UF FLMNH Ichthyology. Version 117.301. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-68"><a href="#cb29-68"></a><span class="co">## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-69"><a href="#cb29-69"></a></span>
<span id="cb29-70"><a href="#cb29-70"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb29-71"><a href="#cb29-71"></a><span class="co">## 'occCite'</span></span>
<span id="cb29-72"><a href="#cb29-72"></a></span>
<span id="cb29-73"><a href="#cb29-73"></a><span class="co">## Writing 6 Bibtex entries ... OK</span></span>
<span id="cb29-74"><a href="#cb29-74"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb29-75"><a href="#cb29-75"></a></span>
<span id="cb29-76"><a href="#cb29-76"></a><span class="co">## Species: Kajikia audax </span></span>
<span id="cb29-77"><a href="#cb29-77"></a><span class="co">## </span></span>
<span id="cb29-78"><a href="#cb29-78"></a><span class="co">## Australian Museum (2021). Australian Museum provider for OZCAM. https://doi.org/10.15468/e7susi. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-79"><a href="#cb29-79"></a><span class="co">## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-80"><a href="#cb29-80"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s (RÃ©union). Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/elttrd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-81"><a href="#cb29-81"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s. Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/gdrknh. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-82"><a href="#cb29-82"></a><span class="co">## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-83"><a href="#cb29-83"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb29-84"><a href="#cb29-84"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb29-85"><a href="#cb29-85"></a><span class="co">## Chiang W (2014). Taiwan Fisheries Research Institute â€“ Digital archives of coastal and offshore specimens. TELDAP. https://doi.org/10.15468/xvxngy. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-86"><a href="#cb29-86"></a><span class="co">## Commonwealth Scientific and Industrial Research Organisation (2020). CSIRO Ichthyology provider for OZCAM. https://doi.org/10.15468/azp1pf. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-87"><a href="#cb29-87"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-88"><a href="#cb29-88"></a><span class="co">## Feeney R (2019). LACM Vertebrate Collection. Version 18.7. Natural History Museum of Los Angeles County. https://doi.org/10.15468/77rmwd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-89"><a href="#cb29-89"></a><span class="co">## Frable B (2019). SIO Marine Vertebrate Collection. Version 1.7. Scripps Institution of Oceanography. https://doi.org/10.15468/ad1ovc. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-90"><a href="#cb29-90"></a><span class="co">## GonzÃ¡lez Acosta A F, ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad C (2020). AmpliaciÃ³n de la base de datos de la ictiofauna insular del Golfo de California. Version 1.7. ComisiÃ³n nacional para el conocimiento y uso de la biodiversidad. https://doi.org/10.15468/p5ovq7. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-91"><a href="#cb29-91"></a><span class="co">## Grant S, McMahan C (2020). Field Museum of Natural History (Zoology) Fish Collection. Version 13.12. Field Museum. https://doi.org/10.15468/alz7wu. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-92"><a href="#cb29-92"></a><span class="co">## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.</span></span>
<span id="cb29-93"><a href="#cb29-93"></a><span class="co">## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.</span></span>
<span id="cb29-94"><a href="#cb29-94"></a><span class="co">## Ministry for Primary Industries (2014). New Zealand research tagging database. Southwestern Pacific OBIS, National Institute of Water and Atmospheric Research (NIWA), Wellington, New Zealand, 411926 records, Online http://nzobisipt.niwa.co.nz/resource.do?r=mpi_tag released on November 5, 2014. https://doi.org/10.15468/i66xdm. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-95"><a href="#cb29-95"></a><span class="co">## Museum and Art Gallery of the Northern Territory (2019). Northern Territory Museum and Art Gallery provider for OZCAM. https://doi.org/10.15468/giro3a. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-96"><a href="#cb29-96"></a><span class="co">## Orrell T, Informatics Office (2021). NMNH Extant Specimen Records. Version 1.41. National Museum of Natural History, Smithsonian Institution. https://doi.org/10.15468/hnhrg3. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-97"><a href="#cb29-97"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb29-98"><a href="#cb29-98"></a><span class="co">## Queensland Museum (2021). Queensland Museum provider for OZCAM. https://doi.org/10.15468/lotsye. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-99"><a href="#cb29-99"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb29-100"><a href="#cb29-100"></a><span class="co">## Robins R (2021). UF FLMNH Ichthyology. Version 117.301. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-101"><a href="#cb29-101"></a><span class="co">## SWPRON (2014). Marine biological observation data from coastal and offshore surveys around New Zealand. Southwestern Pacific OBIS, National Institute of Water and Atmospheric Research (NIWA), Wellington, New Zealand, 9092 records, Online http://nzobisipt.niwa.co.nz/resource.do?r=mbis_nz released on January 16, 2018. https://doi.org/10.15468/pzpgop. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-102"><a href="#cb29-102"></a><span class="co">## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-103"><a href="#cb29-103"></a><span class="co">## Uchifune Y, Yamamoto H (2020). Asia-Pacific Dataset. Version 1.33. National Museum of Nature and Science, Japan. https://doi.org/10.15468/vjeh1p. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-104"><a href="#cb29-104"></a><span class="co">## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-105"><a href="#cb29-105"></a><span class="co">## University of Washington Ichthyology Collection (UWFC) https://doi.org/10.15468/vvp7gr. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-106"><a href="#cb29-106"></a></span>
<span id="cb29-107"><a href="#cb29-107"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb29-108"><a href="#cb29-108"></a><span class="co">## 'occCite'</span></span>
<span id="cb29-109"><a href="#cb29-109"></a></span>
<span id="cb29-110"><a href="#cb29-110"></a><span class="co">## Writing 6 Bibtex entries ... OK</span></span>
<span id="cb29-111"><a href="#cb29-111"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb29-112"><a href="#cb29-112"></a></span>
<span id="cb29-113"><a href="#cb29-113"></a><span class="co">## Species: Tetrapturus angustirostris </span></span>
<span id="cb29-114"><a href="#cb29-114"></a><span class="co">## </span></span>
<span id="cb29-115"><a href="#cb29-115"></a><span class="co">## Australian Museum (2021). Australian Museum provider for OZCAM. https://doi.org/10.15468/e7susi. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-116"><a href="#cb29-116"></a><span class="co">## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-117"><a href="#cb29-117"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s (RÃ©union). Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/elttrd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-118"><a href="#cb29-118"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s. Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/gdrknh. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-119"><a href="#cb29-119"></a><span class="co">## Catania D, Fong J (2021). CAS Ichthyology (ICH). Version 150.260. California Academy of Sciences. https://doi.org/10.15468/efh2ib. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-120"><a href="#cb29-120"></a><span class="co">## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-121"><a href="#cb29-121"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb29-122"><a href="#cb29-122"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb29-123"><a href="#cb29-123"></a><span class="co">## Chiang W (2014). Taiwan Fisheries Research Institute â€“ Digital archives of coastal and offshore specimens. TELDAP. https://doi.org/10.15468/xvxngy. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-124"><a href="#cb29-124"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-125"><a href="#cb29-125"></a><span class="co">## Feeney R (2019). LACM Vertebrate Collection. Version 18.7. Natural History Museum of Los Angeles County. https://doi.org/10.15468/77rmwd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-126"><a href="#cb29-126"></a><span class="co">## Frable B (2019). SIO Marine Vertebrate Collection. Version 1.7. Scripps Institution of Oceanography. https://doi.org/10.15468/ad1ovc. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-127"><a href="#cb29-127"></a><span class="co">## Harvard University M, Morris P J (2021). Museum of Comparative Zoology, Harvard University. Version 162.252. Museum of Comparative Zoology, Harvard University. https://doi.org/10.15468/p5rupv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-128"><a href="#cb29-128"></a><span class="co">## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.</span></span>
<span id="cb29-129"><a href="#cb29-129"></a><span class="co">## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.</span></span>
<span id="cb29-130"><a href="#cb29-130"></a><span class="co">## Ministry for Primary Industries (2014). New Zealand research tagging database. Southwestern Pacific OBIS, National Institute of Water and Atmospheric Research (NIWA), Wellington, New Zealand, 411926 records, Online http://nzobisipt.niwa.co.nz/resource.do?r=mpi_tag released on November 5, 2014. https://doi.org/10.15468/i66xdm. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-131"><a href="#cb29-131"></a><span class="co">## National Museum of Nature and Science, Japan (2020). Fish specimens of Kagoshima University Museum. https://doi.org/10.15468/vcj3j8. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-132"><a href="#cb29-132"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb29-133"><a href="#cb29-133"></a><span class="co">## Queensland Museum (2021). Queensland Museum provider for OZCAM. https://doi.org/10.15468/lotsye. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-134"><a href="#cb29-134"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb29-135"><a href="#cb29-135"></a><span class="co">## Raiva R, Santana P (2019). Diversidade e ocorrÃªncia de peixes em Inhambane (2009-2017). Version 1.4. National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/4fj2tq. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-136"><a href="#cb29-136"></a><span class="co">## Raiva R, Viador R, Santana P (2019). Diversidade e ocorrÃªncia de peixes na ZambÃ©zia (2003-2016). National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/mrz36h. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-137"><a href="#cb29-137"></a><span class="co">## Robins R (2021). UF FLMNH Ichthyology. Version 117.301. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-138"><a href="#cb29-138"></a><span class="co">## South Australian Museum (2021). South Australian Museum Australia provider for OZCAM. https://doi.org/10.15468/wz4rrh. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-139"><a href="#cb29-139"></a><span class="co">## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-140"><a href="#cb29-140"></a><span class="co">## Uchifune Y, Yamamoto H (2020). Asia-Pacific Dataset. Version 1.33. National Museum of Nature and Science, Japan. https://doi.org/10.15468/vjeh1p. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-141"><a href="#cb29-141"></a><span class="co">## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-142"><a href="#cb29-142"></a><span class="co">## Western Australian Museum (2019). Western Australian Museum provider for OZCAM. https://doi.org/10.15468/5qt0dm. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-143"><a href="#cb29-143"></a></span>
<span id="cb29-144"><a href="#cb29-144"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb29-145"><a href="#cb29-145"></a><span class="co">## 'occCite'</span></span>
<span id="cb29-146"><a href="#cb29-146"></a></span>
<span id="cb29-147"><a href="#cb29-147"></a><span class="co">## Writing 6 Bibtex entries ... OK</span></span>
<span id="cb29-148"><a href="#cb29-148"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb29-149"><a href="#cb29-149"></a></span>
<span id="cb29-150"><a href="#cb29-150"></a><span class="co">## Species: Tetrapturus belone </span></span>
<span id="cb29-151"><a href="#cb29-151"></a><span class="co">## </span></span>
<span id="cb29-152"><a href="#cb29-152"></a><span class="co">## Bentley A (2021). KUBI Ichthyology Collection. Version 17.70. University of Kansas Biodiversity Institute. https://doi.org/10.15468/mgjasg. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-153"><a href="#cb29-153"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb29-154"><a href="#cb29-154"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb29-155"><a href="#cb29-155"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-156"><a href="#cb29-156"></a><span class="co">## Harvard University M, Morris P J (2021). Museum of Comparative Zoology, Harvard University. Version 162.252. Museum of Comparative Zoology, Harvard University. https://doi.org/10.15468/p5rupv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-157"><a href="#cb29-157"></a><span class="co">## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.</span></span>
<span id="cb29-158"><a href="#cb29-158"></a><span class="co">## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.</span></span>
<span id="cb29-159"><a href="#cb29-159"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb29-160"><a href="#cb29-160"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb29-161"><a href="#cb29-161"></a><span class="co">## Ranz J (2017). Banco de Datos de la Biodiversidad de la Comunitat Valenciana. Biodiversity data bank of Generalitat Valenciana. https://doi.org/10.15468/b4yqdy. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-162"><a href="#cb29-162"></a><span class="co">## Robins R (2021). UF FLMNH Ichthyology. Version 117.301. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-163"><a href="#cb29-163"></a><span class="co">## UMS PatriNat (OFB-CNRS-MNHN), Paris (2020). DonnÃ©es d'occurrences EspÃ¨ces issues de l'inventaire des ZNIEFF. Version 1.1. https://doi.org/10.15468/ikshke. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-164"><a href="#cb29-164"></a></span>
<span id="cb29-165"><a href="#cb29-165"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb29-166"><a href="#cb29-166"></a><span class="co">## 'occCite'</span></span>
<span id="cb29-167"><a href="#cb29-167"></a></span>
<span id="cb29-168"><a href="#cb29-168"></a><span class="co">## Writing 6 Bibtex entries ... OK</span></span>
<span id="cb29-169"><a href="#cb29-169"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb29-170"><a href="#cb29-170"></a></span>
<span id="cb29-171"><a href="#cb29-171"></a><span class="co">## Species: Tetrapturus georgii </span></span>
<span id="cb29-172"><a href="#cb29-172"></a><span class="co">## </span></span>
<span id="cb29-173"><a href="#cb29-173"></a><span class="co">## Bentley A (2021). KUBI Ichthyology Collection. Version 17.70. University of Kansas Biodiversity Institute. https://doi.org/10.15468/mgjasg. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-174"><a href="#cb29-174"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb29-175"><a href="#cb29-175"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb29-176"><a href="#cb29-176"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-177"><a href="#cb29-177"></a><span class="co">## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.</span></span>
<span id="cb29-178"><a href="#cb29-178"></a><span class="co">## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.</span></span>
<span id="cb29-179"><a href="#cb29-179"></a><span class="co">## Orrell T, Informatics Office (2021). NMNH Extant Specimen Records. Version 1.41. National Museum of Natural History, Smithsonian Institution. https://doi.org/10.15468/hnhrg3. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-180"><a href="#cb29-180"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb29-181"><a href="#cb29-181"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb29-182"><a href="#cb29-182"></a><span class="co">## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-183"><a href="#cb29-183"></a></span>
<span id="cb29-184"><a href="#cb29-184"></a><span class="co">## Warning in utils::citation(pkg): no date field in DESCRIPTION file of package</span></span>
<span id="cb29-185"><a href="#cb29-185"></a><span class="co">## 'occCite'</span></span>
<span id="cb29-186"><a href="#cb29-186"></a></span>
<span id="cb29-187"><a href="#cb29-187"></a><span class="co">## Writing 6 Bibtex entries ... OK</span></span>
<span id="cb29-188"><a href="#cb29-188"></a><span class="co">## Results written to file 'temp.bib'</span></span>
<span id="cb29-189"><a href="#cb29-189"></a></span>
<span id="cb29-190"><a href="#cb29-190"></a><span class="co">## Species: Tetrapturus pfluegeri </span></span>
<span id="cb29-191"><a href="#cb29-191"></a><span class="co">## </span></span>
<span id="cb29-192"><a href="#cb29-192"></a><span class="co">## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-193"><a href="#cb29-193"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s (RÃ©union). Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/elttrd. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-194"><a href="#cb29-194"></a><span class="co">## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s. Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/gdrknh. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-195"><a href="#cb29-195"></a><span class="co">## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-196"><a href="#cb29-196"></a><span class="co">## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.5.2. https://CRAN.R-project.org/package=rgbif.</span></span>
<span id="cb29-197"><a href="#cb29-197"></a><span class="co">## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.</span></span>
<span id="cb29-198"><a href="#cb29-198"></a><span class="co">## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-199"><a href="#cb29-199"></a><span class="co">## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.</span></span>
<span id="cb29-200"><a href="#cb29-200"></a><span class="co">## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.</span></span>
<span id="cb29-201"><a href="#cb29-201"></a><span class="co">## Owens, H.L., Merow, C., Maitner, B., Kass, J.M., Barve, V., Guralnick, R.P. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.4.6. https://github.com/hannahlowens/occCite.</span></span>
<span id="cb29-202"><a href="#cb29-202"></a><span class="co">## R Core Team. (2020). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.</span></span>
<span id="cb29-203"><a href="#cb29-203"></a><span class="co">## Robins R (2021). UF FLMNH Ichthyology. Version 117.301. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</span></span>
<span id="cb29-204"><a href="#cb29-204"></a><span class="co">## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</span></span></code></pre></div>
<hr>
</div>
</div>
<div id="visualization-features" class="section level1">
<h1 class="hasAnchor">
<a href="#visualization-features" class="anchor"></a>Visualization features</h1>
<div id="search-result-summary-figures" class="section level2">
<h2 class="hasAnchor">
<a href="#search-result-summary-figures" class="anchor"></a>Search result summary figures</h2>
<p>occCite includes a <code>plot</code> method for <code>occCiteData</code> objects, which is capable of generating three types of plots. First, letâ€™s see how the results of our multiple species query looks.</p>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span>, 
               bySpecies <span class="op">=</span> <span class="cn">F</span>, 
               plotTypes <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"yearHistogram"</span>, <span class="st">"source"</span>, <span class="st">"aggregator"</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<p><img src="README_files/figure-markdown_github/summary%20figures%20overall-1.png" width="33%"><img src="README_files/figure-markdown_github/summary%20figures%20overall-2.png" width="33%"><img src="README_files/figure-markdown_github/summary%20figures%20overall-3.png" width="33%"></p>
<p>We can also generate plots for each species in the <code>occCiteData</code> object individually. Since GBIF is the only aggregator we used for the query, Iâ€™ll skip generating the aggregator plot.</p>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span>, 
               bySpecies <span class="op">=</span> <span class="cn">T</span>, 
               plotTypes <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"yearHistogram"</span>, <span class="st">"source"</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<p><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-1.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-2.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-3.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-4.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-5.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-6.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-7.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-8.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-9.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-10.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-11.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-12.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-13.png" width="50%"><img src="README_files/figure-markdown_github/summary%20figures%20by%20species-14.png" width="50%"></p>
</div>
<div id="mapping-occcite-search-results" class="section level2">
<h2 class="hasAnchor">
<a href="#mapping-occcite-search-results" class="anchor"></a>Mapping occCite search results</h2>
<p>occCite also contains tools to map search results. We can pass occCite objects to be mapped as an interactive html widget using <code>leaflet</code>. If there are multiple species in a single occCite object, occurrences will be color-coded by species. If you click on a single occurrence, you will be able to see the species, the coordinates, the date on which it was collected, the dataset to which it belongs, and the dataservice that provided the record.</p>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="reference/occCiteMap.html">occCiteMap</a></span><span class="op">(</span><span class="va">mySimpleOccCiteObject</span>, cluster <span class="op">=</span> <span class="cn">T</span><span class="op">)</span></code></pre></div>
</div>
</div>

  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2>Links</h2>
<ul class="list-unstyled">
<li>Download from CRAN at <br><a href="https://cloud.r-project.org/package=occCite">https://â€‹cloud.r-project.org/â€‹package=occCite</a>
</li>
<li>Browse source code at <br><a href="https://github.com/hannahlowens/occCite/">https://â€‹github.com/â€‹hannahlowens/â€‹occCite/â€‹</a>
</li>
<li>Report a bug at <br><a href="https://github.com/hannahlowens/occCite/issues">https://â€‹github.com/â€‹hannahlowens/â€‹occCite/â€‹issues</a>
</li>
</ul>
</div>
<div class="license">
<h2>License</h2>
<ul class="list-unstyled">
<li><a href="LICENSE.html">Full license</a></li>
<li><small><a href="https://www.r-project.org/Licenses/GPL-3">GPL-3</a></small></li>
</ul>
</div>
<div class="developers">
<h2>Developers</h2>
<ul class="list-unstyled">
<li>Hannah L. Owens <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0003-0071-1745" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Cory Merow <br><small class="roles"> Author </small> <a href="https://orcid.org/0000-0003-0561-053X" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Brian Maitner <br><small class="roles"> Author </small> <a href="https://orcid.org/0000-0002-2118-9880" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Jamie M. Kass <br><small class="roles"> Author </small> <a href="https://orcid.org/0000-0002-9432-895X" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Vijay Barve <br><small class="roles"> Author </small> <a href="https://orcid.org/0000-0002-4852-2567" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Robert P. Guralnick <br><small class="roles"> Author </small> <a href="https://orcid.org/0000-0001-6682-1504" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>

  <div class="dev-status">
<h2>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://github.com/hannahlowens/occCite/actions"><img src="https://github.com/hannahlowens/occCite/workflows/R-CMD-check/badge.svg" alt="R build status"></a></li>
<li><a href="https://cran.r-project.org/package=occCite"><img src="https://www.r-pkg.org/badges/version/occCite" alt="cran version"></a></li>
<li><a href="https://github.com/r-hub/cranlogs.app"><img src="https://cranlogs.r-pkg.org/badges/occCite" alt="rstudio mirror downloads"></a></li>
</ul>
</div>
</div>
</div>


      <footer><div class="copyright">
  <p>Developed by Hannah L. Owens, Cory Merow, Brian Maitner, Jamie M. Kass, Vijay Barve, Robert P. Guralnick.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
