<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Advanced execution of occurrence searches and citations generation â€¢ occCite</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Advanced execution of occurrence searches and citations generation">
<meta property="og:description" content="occCite">
<meta property="og:image" content="/reference/figures/occHex.png">
<meta property="og:image:alt" content="Hex logo of a museum map icon with 'occCite' written below it">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:creator" content="@hannahOish">
<meta name="twitter:site" content="@hannahOish">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">occCite</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.5.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Advanced.html">Advanced execution of occurrence searches and citations generation</a>
    </li>
    <li>
      <a href="../articles/Simple.html">Simple execution of occurrence searches and citation generation</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/hannahlowens/occCite/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="Advanced_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Advanced execution of occurrence searches and citations generation</h1>
                        <h4 class="author">Hannah L. Owens</h4>
                        <h4 class="author">Cory Merow</h4>
                        <h4 class="author">Brian Maitner</h4>
                        <h4 class="author">Jamie M. Kass</h4>
                        <h4 class="author">Vijay Barve</h4>
                        <h4 class="author">Robert Guralnick</h4>
            
            <h4 class="date">2021-10-14</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/hannahlowens/occCite/blob/master/vignettes/Advanced.Rmd"><code>vignettes/Advanced.Rmd</code></a></small>
      <div class="hidden name"><code>Advanced.Rmd</code></div>

    </div>

    
    
<div id="advanced-features" class="section level1">
<h1 class="hasAnchor">
<a href="#advanced-features" class="anchor"></a>Advanced features</h1>
<p>This vignette demonstrates more advanced features and customization available in <code>occCite</code>. We recommend you read <code>vignette("Simple.Rmd", package = "occCite")</code> first, if you have not already done so.</p>
<div id="loading-data-from-previous-gbif-searches" class="section level2">
<h2 class="hasAnchor">
<a href="#loading-data-from-previous-gbif-searches" class="anchor"></a>Loading data from previous GBIF searches</h2>
<p>Querying GBIF can take quite a bit of time, especially for multiple species and/or well-known species. In this case, you may wish to access previously-downloaded data sets from your computer by specifying the general location of your downloaded <code>.zip</code> files. <code>occQuery</code> will crawl through your specified <code>GBIFDownloadDirectory</code> to collect all the <code>.zip</code> files contained in that folder and its subfolders. It will then import the most recent downloads that match your taxon list. These GBIF data will be appended to a BIEN search the same as if you do the simple real-time search (if you chose BIEN as well as GBIF), as was shown above. <code>checkPreviousGBIFDownload</code> is <code>TRUE</code> by default, but if <code>loadLocalGBIFDownload</code> is <code>TRUE</code>, <code>occQuery</code> will ignore <code>checkPreviousDownload</code>. It is also worth noting that <code>occCite</code> does not currently support mixed data download sources. That is, you cannot do GBIF queries for some taxa, download previously-prepared data sets for others, and load the rest from local data sets on your computer.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Simple load</span>
<span class="va">myOldOccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/occQuery.html">occQuery</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Protea cynaroides"</span>, 
                                  datasources <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"gbif"</span>, <span class="st">"bien"</span><span class="op">)</span>, 
                                  GBIFLogin <span class="op">=</span> <span class="cn">NULL</span>, 
                                  GBIFDownloadDirectory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span><span class="op">(</span><span class="st">'extdata/'</span>, package<span class="op">=</span><span class="st">'occCite'</span><span class="op">)</span>,
                                  loadLocalGBIFDownload <span class="op">=</span> <span class="cn">T</span>,
                                  checkPreviousGBIFDownload <span class="op">=</span> <span class="cn">F</span><span class="op">)</span></code></pre></div>
<p>Here is the result. Look familiar?</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#GBIF search results</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">myOldOccCiteObject</span><span class="op">@</span><span class="va">occResults</span><span class="op">$</span><span class="va">`Protea cynaroides`</span><span class="op">$</span><span class="va">GBIF</span><span class="op">$</span><span class="va">OccurrenceTable</span><span class="op">)</span>;</code></pre></div>
<pre><code>##                name longitude  latitude day month year
## 1 Protea cynaroides  18.40540 -33.95891   4     1 2015
## 2 Protea cynaroides  18.42350 -33.96619  20     6 2019
## 3 Protea cynaroides  22.99340 -34.05478  16     6 2019
## 4 Protea cynaroides  18.40232 -34.08405   9     6 2019
## 5 Protea cynaroides  19.44807 -34.52123  13     6 2019
## 6 Protea cynaroides  18.39757 -34.07418   9     6 2019
##                                   Dataset                           DatasetKey
## 1 iNaturalist Research-grade Observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7
## 2 iNaturalist Research-grade Observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7
## 3 iNaturalist Research-grade Observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7
## 4 iNaturalist Research-grade Observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7
## 5 iNaturalist Research-grade Observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7
## 6 iNaturalist Research-grade Observations 50c9509d-22c7-4a22-a47d-8c48425ef4a7
##   DataService
## 1        GBIF
## 2        GBIF
## 3        GBIF
## 4        GBIF
## 5        GBIF
## 6        GBIF</code></pre>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#The full summary</span>
<span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">myOldOccCiteObject</span><span class="op">)</span></code></pre></div>
<pre><code>##  
##  OccCite query occurred on: 14 October, 2021
##  
##  User query type: User-supplied list of taxa.
##  
##  Sources for taxonomic rectification: GBIF Backbone Taxonomy
##      
##  
##  Taxonomic cleaning results:     
## 
##          Input Name                        Best Match
## 1 Protea cynaroides Protea cynaroides (L.) L. L. (L.)
##   Taxonomic Databases w/ Matches
## 1         GBIF Backbone Taxonomy
##  
##  Sources for occurrence data: gbif, bien
##      
##                             Species Occurrences Sources
## 1 Protea cynaroides (L.) L. L. (L.)         825      14
##  
##  GBIF dataset DOIs:  
## 
##                             Species GBIF Access Date           GBIF DOI
## 1 Protea cynaroides (L.) L. L. (L.)       2019-07-15 10.15468/dl.iqnra2</code></pre>
<p>Getting citation data works the exact same way with previously-downloaded data as it does from a fresh data set.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Get citations</span>
<span class="va">myOldOccCitations</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/occCitation.html">occCitation</a></span><span class="op">(</span><span class="va">myOldOccCiteObject</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">myOldOccCitations</span><span class="op">)</span></code></pre></div>
<pre><code>## Writing 5 Bibtex entries ... OK
## Results written to file 'temp.bib'</code></pre>
<pre><code>## Warning: `as_data_frame()` was deprecated in tibble 2.0.0.
## Please use `as_tibble()` instead.
## The signature and semantics have changed, see `?as_tibble`.
## This warning is displayed once every 8 hours.
## Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated.</code></pre>
<pre><code>## Cameron E, Auckland Museum A M (2021). Auckland Museum Botany Collection. Version 1.65. Auckland War Memorial Museum. https://doi.org/10.15468/mnjkvv. Accessed via GBIF on 2019-07-15.
## Capers R (2014). CONN. University of Connecticut. https://doi.org/10.15468/w35jmd. Accessed via GBIF on 2019-07-15.
## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.6.0. https://CRAN.R-project.org/package=rgbif.
## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.
## Fatima Parker-Allie, Ranwashe F (2018). PRECIS. South African National Biodiversity Institute. https://doi.org/10.15468/rckmn2. Accessed via GBIF on 2019-07-15.
## Maitner, B. (2020). BIEN: Tools for Accessing the Botanical Information and Ecology. R package version 1.2.4. https://CRAN.R-project.org/package=BIEN.
## Missouri Botanical Garden,Herbarium. Accessed via BIEN on NA.
## MNHN, Chagnoux S (2021). The vascular plants collection (P) at the Herbarium of the MusÃ©um national d'Histoire Naturelle (MNHN - Paris). Version 69.231. MNHN - Museum national d'Histoire naturelle. https://doi.org/10.15468/nc6rxy. Accessed via GBIF on 2019-07-15.
## MNHN. Accessed via BIEN on NA.
## naturgucker.de. naturgucker. https://doi.org/10.15468/uc1apo. Accessed via GBIF on 2019-07-15.
## Owens, H., Merow, C., Maitner, B., Kass, J., Barve, V., Guralnick, R. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.5.0. https://CRAN.R-project.org/package=occCite.
## R Core Team. (2021). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.
## Ranwashe F (2021). BODATSA: Botanical Collections. Version 1.5. South African National Biodiversity Institute. https://doi.org/10.15468/2aki0q. Accessed via GBIF on 2019-07-15.
## SANBI. Accessed via BIEN on NA.
## Senckenberg (2020). African Plants - a photo guide. https://doi.org/10.15468/r9azth. Accessed via GBIF on 2019-07-15.
## Solomon J, Stimmel H (2021). Tropicos Specimen Data. Missouri Botanical Garden. https://doi.org/10.15468/hja69f. Accessed via GBIF on 2019-07-15.
## Tela Botanica. Carnet en Ligne. https://doi.org/10.15468/rydcn2. Accessed via GBIF on 2019-07-15.
## UConn. Accessed via BIEN on NA.
## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-15.</code></pre>
<p>Note that you can also load multiple species using either a vector of species names or a phylogeny (provided you have previously downloaded data for all of the species of interest), and you can load occurrences from non-GBIF data sources (e.g.Â BIEN) in the same query.</p>
<hr>
</div>
<div id="performing-a-multi-species-search" class="section level2">
<h2 class="hasAnchor">
<a href="#performing-a-multi-species-search" class="anchor"></a>Performing a Multi-Species Search</h2>
<p>In addition to doing a simple, single species search, you can also use <code>occCite</code> to search for and manage occurrence datasets for multiple species. You can either submit a vector of species names, or you can submit a <em>phylogeny</em>! The occCitation function will return a named list of citation tables in the case of multiple species.</p>
</div>
<div id="occcite-with-a-phylogeny" class="section level2">
<h2 class="hasAnchor">
<a href="#occcite-with-a-phylogeny" class="anchor"></a>occCite with a Phylogeny</h2>
<p>Here is an example of how such a search is structured, using an unpublished phylogeny of billfishes.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="http://ape-package.ird.fr/">ape</a></span><span class="op">)</span>
<span class="co">#Get tree</span>
<span class="va">treeFile</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span><span class="op">(</span><span class="st">"extdata/Fish_12Tax_time_calibrated.tre"</span>, package<span class="op">=</span><span class="st">'occCite'</span><span class="op">)</span>
<span class="va">phylogeny</span> <span class="op">&lt;-</span> <span class="fu">ape</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ape/man/read.nexus.html">read.nexus</a></span><span class="op">(</span><span class="va">treeFile</span><span class="op">)</span>
<span class="va">tree</span> <span class="op">&lt;-</span> <span class="fu">ape</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ape/man/drop.tip.html">extract.clade</a></span><span class="op">(</span><span class="va">phylogeny</span>, <span class="fl">21</span><span class="op">)</span>
<span class="co">#Query databases for names</span>
<span class="va">myPhyOccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/studyTaxonList.html">studyTaxonList</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">tree</span>, 
                                     datasources <span class="op">=</span> <span class="st">"GBIF Backbone Taxonomy"</span><span class="op">)</span>
<span class="co">#Query GBIF for occurrence data</span>
<span class="va">myPhyOccCiteObject</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/occQuery.html">occQuery</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">myPhyOccCiteObject</span>, 
                            datasources <span class="op">=</span> <span class="st">"gbif"</span>,
                            GBIFDownloadDirectory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span><span class="op">(</span><span class="st">'extdata/'</span>, package<span class="op">=</span><span class="st">'occCite'</span><span class="op">)</span>,
                            loadLocalGBIFDownload <span class="op">=</span> <span class="cn">T</span>,
                            checkPreviousGBIFDownload <span class="op">=</span> <span class="cn">F</span><span class="op">)</span>
<span class="co"># What does a multispecies query look like?</span>
<span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span><span class="op">)</span></code></pre></div>
<pre><code>##  
##  OccCite query occurred on: 14 October, 2021
##  
##  User query type: User-supplied phylogeny.
##  
##  Sources for taxonomic rectification: GBIF Backbone Taxonomy
##      
##  
##  Taxonomic cleaning results:     
## 
##                   Input Name                                    Best Match
## 1 Tetrapturus_angustirostris       Tetrapturus angustirostris Tanaka, 1915
## 2         Tetrapturus_belone           Tetrapturus belone Rafinesque, 1810
## 3        Tetrapturus_georgii                Tetrapturus georgii Lowe, 1841
## 4      Tetrapturus_pfluegeri Tetrapturus pfluegeri Robins &amp; de Sylva, 1963
##   Taxonomic Databases w/ Matches
## 1         GBIF Backbone Taxonomy
## 2         GBIF Backbone Taxonomy
## 3         GBIF Backbone Taxonomy
## 4         GBIF Backbone Taxonomy
##  
##  Sources for occurrence data: gbif
##      
##                                         Species Occurrences Sources
## 1       Tetrapturus angustirostris Tanaka, 1915         649      23
## 2           Tetrapturus belone Rafinesque, 1810           9       6
## 3                Tetrapturus georgii Lowe, 1841          62       4
## 4 Tetrapturus pfluegeri Robins &amp; de Sylva, 1963         410       8
##  
##  GBIF dataset DOIs:  
## 
##                                         Species GBIF Access Date
## 1       Tetrapturus angustirostris Tanaka, 1915       2019-07-04
## 2           Tetrapturus belone Rafinesque, 1810       2019-07-04
## 3                Tetrapturus georgii Lowe, 1841       2019-07-04
## 4 Tetrapturus pfluegeri Robins &amp; de Sylva, 1963       2019-07-04
##             GBIF DOI
## 1 10.15468/dl.mumi5e
## 2 10.15468/dl.q2nxb1
## 3 10.15468/dl.h860up
## 4 10.15468/dl.qjidbs</code></pre>
<p>When you have results for multiple species, as in this case, you can also plot the summary figures either for the whole searchâ€¦</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span><span class="op">)</span></code></pre></div>
<p><img src="Advanced_files/figure-html/plotting%20all%20species-1.png" width="100%"><img src="Advanced_files/figure-html/plotting%20all%20species-2.png" width="100%"><img src="Advanced_files/figure-html/plotting%20all%20species-3.png" width="100%"></p>
<p><em>or</em> you can plot the results by species!</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span>, bySpecies <span class="op">=</span> <span class="cn">T</span>, plotTypes <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"yearHistogram"</span>, <span class="st">"source"</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<p><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-1.png" width="100%"><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-2.png" width="100%"><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-3.png" width="100%"><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-4.png" width="100%"><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-5.png" width="100%"><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-6.png" width="100%"><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-7.png" width="100%"><img src="Advanced_files/figure-html/plotting%20phylogenetic%20search%20by%20species-8.png" width="100%"></p>
<p>And then you can print out the citations, separated by species (or not, but in this example, theyâ€™re separate).</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Get citations</span>
<span class="va">myPhyOccCitations</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/occCitation.html">occCitation</a></span><span class="op">(</span><span class="va">myPhyOccCiteObject</span><span class="op">)</span>

<span class="co">#Print citations as text with accession dates.</span>
<span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">myPhyOccCitations</span>, bySpecies <span class="op">=</span> <span class="cn">T</span><span class="op">)</span></code></pre></div>
<pre><code>## Writing 6 Bibtex entries ... OK
## Results written to file 'temp.bib'</code></pre>
<pre><code>## Species: Tetrapturus angustirostris Tanaka, 1915 
## 
## Australian Museum (2021). Australian Museum provider for OZCAM. https://doi.org/10.15468/e7susi. Accessed via GBIF on 2019-07-04.
## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.
## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s (RÃ©union). Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/elttrd. Accessed via GBIF on 2019-07-04.
## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s. Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/gdrknh. Accessed via GBIF on 2019-07-04.
## Bureau of Rural Sciences - National commercial fisheries half-degree data set 2000-2002 https://doi.org/10.15468/0esdv0. Accessed via GBIF on 2019-07-04.
## Catania D, Fong J (2021). CAS Ichthyology (ICH). Version 150.290. California Academy of Sciences. https://doi.org/10.15468/efh2ib. Accessed via GBIF on 2019-07-04.
## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.
## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.6.0. https://CRAN.R-project.org/package=rgbif.
## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.
## Chiang W (2014). Taiwan Fisheries Research Institute â€“ Digital archives of coastal and offshore specimens. TELDAP. https://doi.org/10.15468/xvxngy. Accessed via GBIF on 2019-07-04.
## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.
## Frable B (2019). SIO Marine Vertebrate Collection. Version 1.7. Scripps Institution of Oceanography. https://doi.org/10.15468/ad1ovc. Accessed via GBIF on 2019-07-04.
## Harvard University M, Morris P J (2021). Museum of Comparative Zoology, Harvard University. Version 162.282. Museum of Comparative Zoology, Harvard University. https://doi.org/10.15468/p5rupv. Accessed via GBIF on 2019-07-04.
## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.
## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.
## Mertz W (2021). LACM Vertebrate Collection. Version 18.9. Natural History Museum of Los Angeles County. https://doi.org/10.15468/77rmwd. Accessed via GBIF on 2019-07-04.
## Ministry for Primary Industries (2014). New Zealand research tagging database. Southwestern Pacific OBIS, National Institute of Water and Atmospheric Research (NIWA), Wellington, New Zealand, 411926 records, Online http://nzobisipt.niwa.co.nz/resource.do?r=mpi_tag released on November 5, 2014. https://doi.org/10.15468/i66xdm. Accessed via GBIF on 2019-07-04.
## National Museum of Nature and Science, Japan (2021). Fish specimens of Kagoshima University Museum. https://doi.org/10.15468/vcj3j8. Accessed via GBIF on 2019-07-04.
## Owens, H., Merow, C., Maitner, B., Kass, J., Barve, V., Guralnick, R. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.5.0. https://CRAN.R-project.org/package=occCite.
## Queensland Museum (2021). Queensland Museum provider for OZCAM. https://doi.org/10.15468/lotsye. Accessed via GBIF on 2019-07-04.
## R Core Team. (2021). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.
## Raiva R, Santana P (2021). Diversidade e ocorrÃªncia de peixes em Inhambane (2009-2017). Version 1.7. National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/4fj2tq. Accessed via GBIF on 2019-07-04.
## Raiva R, Viador R, Santana P (2021). Diversidade e ocorrÃªncia de peixes na ZambÃ©zia (2003-2016). Version 1.6. National Institute of Fisheries Research (IIP) â€“ Mozambique. https://doi.org/10.15468/mrz36h. Accessed via GBIF on 2019-07-04.
## Robins R (2021). UF FLMNH Ichthyology. Version 117.333. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.
## South Australian Museum (2021). South Australian Museum Australia provider for OZCAM. https://doi.org/10.15468/wz4rrh. Accessed via GBIF on 2019-07-04.
## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.
## Uchifune Y, Yamamoto H (2021). Asia-Pacific Dataset. Version 1.36. National Museum of Nature and Science, Japan. https://doi.org/10.15468/vjeh1p. Accessed via GBIF on 2019-07-04.
## Ueda K (2021). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2019-07-04.
## Western Australian Museum (2019). Western Australian Museum provider for OZCAM. https://doi.org/10.15468/5qt0dm. Accessed via GBIF on 2019-07-04.</code></pre>
<pre><code>## Writing 6 Bibtex entries ... OK
## Results written to file 'temp.bib'</code></pre>
<pre><code>## Species: Tetrapturus belone Rafinesque, 1810 
## 
## Bentley A (2021). KUBI Ichthyology Collection. Version 17.78. University of Kansas Biodiversity Institute. https://doi.org/10.15468/mgjasg. Accessed via GBIF on 2019-07-04.
## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.6.0. https://CRAN.R-project.org/package=rgbif.
## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.
## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.
## Harvard University M, Morris P J (2021). Museum of Comparative Zoology, Harvard University. Version 162.282. Museum of Comparative Zoology, Harvard University. https://doi.org/10.15468/p5rupv. Accessed via GBIF on 2019-07-04.
## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.
## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.
## Owens, H., Merow, C., Maitner, B., Kass, J., Barve, V., Guralnick, R. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.5.0. https://CRAN.R-project.org/package=occCite.
## R Core Team. (2021). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.
## Ranz J (2017). Banco de Datos de la Biodiversidad de la Comunitat Valenciana. Biodiversity data bank of Generalitat Valenciana. https://doi.org/10.15468/b4yqdy. Accessed via GBIF on 2019-07-04.
## ROBERT S, LEPAREUR F, Inventaire National du Patrimoine Naturel (2021). DonnÃ©es d'occurrences EspÃ¨ces issues de l'inventaire des ZNIEFF. Version 1.6. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/ikshke. Accessed via GBIF on 2019-07-04.
## Robins R (2021). UF FLMNH Ichthyology. Version 117.333. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.</code></pre>
<pre><code>## Writing 6 Bibtex entries ... OK
## Results written to file 'temp.bib'</code></pre>
<pre><code>## Species: Tetrapturus georgii Lowe, 1841 
## 
## Bentley A (2021). KUBI Ichthyology Collection. Version 17.78. University of Kansas Biodiversity Institute. https://doi.org/10.15468/mgjasg. Accessed via GBIF on 2019-07-04.
## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.6.0. https://CRAN.R-project.org/package=rgbif.
## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.
## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.
## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.
## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.
## Orrell T, Informatics Office (2021). NMNH Extant Specimen Records. Version 1.46. National Museum of Natural History, Smithsonian Institution. https://doi.org/10.15468/hnhrg3. Accessed via GBIF on 2019-07-04.
## Owens, H., Merow, C., Maitner, B., Kass, J., Barve, V., Guralnick, R. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.5.0. https://CRAN.R-project.org/package=occCite.
## R Core Team. (2021). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.
## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</code></pre>
<pre><code>## Writing 6 Bibtex entries ... OK
## Results written to file 'temp.bib'</code></pre>
<pre><code>## Species: Tetrapturus pfluegeri Robins &amp; de Sylva, 1963 
## 
## Barde J (2011). ecoscope_observation_database. IRD - Institute of Research for Development. https://doi.org/10.15468/dz1kk0. Accessed via GBIF on 2019-07-04.
## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s (RÃ©union). Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/elttrd. Accessed via GBIF on 2019-07-04.
## BARDE Julien N, Inventaire National du Patrimoine Naturel (2019). Programme Ecoscope: donnÃ©es d'observations des Ã©cosystÃ¨mes marins exploitÃ©s. Version 1.1. UMS PatriNat (OFB-CNRS-MNHN), Paris. https://doi.org/10.15468/gdrknh. Accessed via GBIF on 2019-07-04.
## Boateng M (2021). Fishes of Ghana. Version 1.4. Department of Marine and Fisheries Sciences, University of Ghana. https://doi.org/10.15468/pgesnw. Accessed via GBIF on 2019-07-04.
## Cauquil P, Barde J (2011). observe_tuna_bycatch_ecoscope. IRD - Institute of Research for Development. https://doi.org/10.15468/23m361. Accessed via GBIF on 2019-07-04.
## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2021). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.6.0. https://CRAN.R-project.org/package=rgbif.
## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.
## European Nucleotide Archive (EMBL-EBI) (2019). Geographically tagged INSDC sequences. https://doi.org/10.15468/cndomv. Accessed via GBIF on 2019-07-04.
## McLean, M.W. (2014). Straightforward Bibliography Management in R Using the RefManager Package. NA, NA. https://arxiv.org/abs/1403.2036.
## McLean, M.W. (2017). RefManageR: Import and Manage BibTeX and BibLaTeX References in R. The Journal of Open Source Software.
## Owens, H., Merow, C., Maitner, B., Kass, J., Barve, V., Guralnick, R. (2021). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.5.0. https://CRAN.R-project.org/package=occCite.
## R Core Team. (2021). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.
## Robins R (2021). UF FLMNH Ichthyology. Version 117.333. Florida Museum of Natural History. https://doi.org/10.15468/8mjsel. Accessed via GBIF on 2019-07-04.
## The International Barcode of Life Consortium (2016). International Barcode of Life project (iBOL). https://doi.org/10.15468/inygc6. Accessed via GBIF on 2019-07-04.</code></pre>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Hannah L. Owens, Cory Merow, Brian Maitner, Jamie M. Kass, Vijay Barve, Robert P. Guralnick.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
